<?xml version="1.0" encoding="UTF-8"?><testsuite name="nosetests" tests="158" errors="19" failures="0" skip="3"><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_dashboard" time="2.075"><error type="exceptions.AttributeError" message="assert_called_once&#10;-------------------- &gt;&gt; begin captured logging &lt;&lt; --------------------&#10;eulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin&#10;requests.packages.urllib3.connectionpool: INFO: Starting new HTTPS connection (1): libfeddev1.library.emory.edu&#10;py.warnings: WARNING: /Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/requests/packages/urllib3/util/ssl_.py:90: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.&#10;  InsecurePlatformWarning&#10;&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;GET /fedora/objects?query=pid~oe-zo-test%3A%2A&amp;pid=true&amp;resultFormat=xml HTTP/1.1&quot; 200 None&#10;eulfedora.api: DEBUG: GET objects=&gt;200: 0.091484 sec&#10;eulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin&#10;eulfedora.models: DEBUG: openemory.common.fedora.DigitalObject has no content models&#10;eulfedora.models: DEBUG: cmodel for openemory.publication.models.Article is info:fedora/emory-control:PublishedArticle-1.0&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;GET /fedora/objects/emory-control:PublishedArticle-1.0?format=xml HTTP/1.1&quot; 200 None&#10;eulfedora.api: DEBUG: GET objects/emory-control:PublishedArticle-1.0=&gt;200: 0.033169 sec&#10;eulfedora.models: DEBUG: info:fedora/emory-control:PublishedArticle-1.0 already exists&#10;eulfedora.models: DEBUG: cmodel for eulfedora.models.ContentModel is info:fedora/fedora-system:ContentModel-3.0&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;GET /fedora/objects/fedora-system:ContentModel-3.0?format=xml HTTP/1.1&quot; 200 None&#10;eulfedora.api: DEBUG: GET objects/fedora-system:ContentModel-3.0=&gt;200: 0.033224 sec&#10;eulfedora.models: DEBUG: info:fedora/fedora-system:ContentModel-3.0 already exists&#10;eulfedora.models: DEBUG: eulfedora.models.DigitalObject has no content models&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/objects/new?logMessage=loaded+from+fixture HTTP/1.1&quot; 500 171&#10;eulfedora.api: DEBUG: POST objects/new=&gt;500: 0.050677 sec&#10;eulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin&#10;py.warnings: WARNING: /Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead&#10;  DeprecationWarning)&#10;&#10;pidservices.clients: DEBUG: Request: POST https://testpid.library.emory.edu/ark/ {'Content-Length': 3, 'Content-type': 'application/x-www-form-urlencoded', 'Accept': 'text/plain'} &lt;![BODY[{'domain': 'http://testpid.library.emory.edu/domains/34/', 'name': 'A very scholarly article', 'target_uri': u'http://example.com/publications/oe-zo-test:{%PID%}/'}]]&gt;&#10;requests.packages.urllib3.connectionpool: INFO: Starting new HTTPS connection (1): testpid.library.emory.edu&#10;py.warnings: WARNING: /Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/requests/packages/urllib3/util/ssl_.py:90: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.&#10;  InsecurePlatformWarning&#10;&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /ark/ HTTP/1.1&quot; 201 None&#10;requests.packages.urllib3.connectionpool: INFO: Resetting dropped connection: libfeddev1.library.emory.edu&#10;py.warnings: WARNING: /Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/requests/packages/urllib3/util/ssl_.py:90: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.&#10;  InsecurePlatformWarning&#10;&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.064002 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.014030 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.017593 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.013960 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/objects/new HTTP/1.1&quot; 201 16&#10;eulfedora.api: DEBUG: POST objects/new=&gt;201: 0.488926 sec&#10;tracking.middleware: INFO: Updating untracked user agent cache&#10;tracking.middleware: DEBUG: Created a new visitor: {'session_key': u'127.0.0.1:', 'ip_address': '127.0.0.1'}&#10;tracking.middleware: DEBUG: Using existing visitor for IP 127.0.0.1 / UA : 1&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;DELETE /fedora/objects/oe-zo-test:pfxjk HTTP/1.1&quot; 200 24&#10;eulfedora.api: DEBUG: DELETE objects/oe-zo-test:pfxjk=&gt;200: 0.308455 sec&#10;--------------------- &gt;&gt; end captured logging &lt;&lt; ---------------------"><![CDATA[  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 67, in testPartExecutor
    yield
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 625, in run
    testMethod()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/Users/azotov/OpenEmory/openemory/accounts/tests.py", line 435, in test_dashboard
    mockprofile.recent_articles.assert_called_once()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 721, in __getattr__
    raise AttributeError(name)
'assert_called_once\n-------------------- >> begin captured logging << --------------------\neulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTPS connection (1): libfeddev1.library.emory.edu\npy.warnings: WARNING: /Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/requests/packages/urllib3/util/ssl_.py:90: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.\n  InsecurePlatformWarning\n\nrequests.packages.urllib3.connectionpool: DEBUG: "GET /fedora/objects?query=pid~oe-zo-test%3A%2A&pid=true&resultFormat=xml HTTP/1.1" 200 None\neulfedora.api: DEBUG: GET objects=>200: 0.091484 sec\neulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin\neulfedora.models: DEBUG: openemory.common.fedora.DigitalObject has no content models\neulfedora.models: DEBUG: cmodel for openemory.publication.models.Article is info:fedora/emory-control:PublishedArticle-1.0\nrequests.packages.urllib3.connectionpool: DEBUG: "GET /fedora/objects/emory-control:PublishedArticle-1.0?format=xml HTTP/1.1" 200 None\neulfedora.api: DEBUG: GET objects/emory-control:PublishedArticle-1.0=>200: 0.033169 sec\neulfedora.models: DEBUG: info:fedora/emory-control:PublishedArticle-1.0 already exists\neulfedora.models: DEBUG: cmodel for eulfedora.models.ContentModel is info:fedora/fedora-system:ContentModel-3.0\nrequests.packages.urllib3.connectionpool: DEBUG: "GET /fedora/objects/fedora-system:ContentModel-3.0?format=xml HTTP/1.1" 200 None\neulfedora.api: DEBUG: GET objects/fedora-system:ContentModel-3.0=>200: 0.033224 sec\neulfedora.models: DEBUG: info:fedora/fedora-system:ContentModel-3.0 already exists\neulfedora.models: DEBUG: eulfedora.models.DigitalObject has no content models\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/objects/new?logMessage=loaded+from+fixture HTTP/1.1" 500 171\neulfedora.api: DEBUG: POST objects/new=>500: 0.050677 sec\neulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin\npy.warnings: WARNING: /Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/django/conf/urls/defaults.py:3: DeprecationWarning: django.conf.urls.defaults is deprecated; use django.conf.urls instead\n  DeprecationWarning)\n\npidservices.clients: DEBUG: Request: POST https://testpid.library.emory.edu/ark/ {\'Content-Length\': 3, \'Content-type\': \'application/x-www-form-urlencoded\', \'Accept\': \'text/plain\'} <![BODY[{\'domain\': \'http://testpid.library.emory.edu/domains/34/\', \'name\': \'A very scholarly article\', \'target_uri\': u\'http://example.com/publications/oe-zo-test:{%PID%}/\'}]]>]]&gt;<![CDATA[\nrequests.packages.urllib3.connectionpool: INFO: Starting new HTTPS connection (1): testpid.library.emory.edu\npy.warnings: WARNING: /Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/requests/packages/urllib3/util/ssl_.py:90: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.\n  InsecurePlatformWarning\n\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /ark/ HTTP/1.1" 201 None\nrequests.packages.urllib3.connectionpool: INFO: Resetting dropped connection: libfeddev1.library.emory.edu\npy.warnings: WARNING: /Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/requests/packages/urllib3/util/ssl_.py:90: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. For more information, see https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning.\n  InsecurePlatformWarning\n\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.064002 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.014030 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.017593 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.013960 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/objects/new HTTP/1.1" 201 16\neulfedora.api: DEBUG: POST objects/new=>201: 0.488926 sec\ntracking.middleware: INFO: Updating untracked user agent cache\ntracking.middleware: DEBUG: Created a new visitor: {\'session_key\': u\'127.0.0.1:\', \'ip_address\': \'127.0.0.1\'}\ntracking.middleware: DEBUG: Using existing visitor for IP 127.0.0.1 / UA : 1\nrequests.packages.urllib3.connectionpool: DEBUG: "DELETE /fedora/objects/oe-zo-test:pfxjk HTTP/1.1" 200 24\neulfedora.api: DEBUG: DELETE objects/oe-zo-test:pfxjk=>200: 0.308455 sec\n--------------------- >> end captured logging << ---------------------']]></error></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_dashboard_announcements" time="1.317"></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_dashboard_stats" time="1.573"></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_degree_institutions_autocomplete" time="1.786"></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_edit_profile" time="3.433"></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_faculty_autocomplete" time="1.473"></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_feedback" time="1.822"></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_get_profile_user" time="1.444"></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_grant_autocomplete" time="1.386"></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_interests_autocomplete" time="1.537"></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_list_departments" time="1.422"></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_login" time="1.791"></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_position_autocomplete" time="1.533"></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_profile" time="0.000"><skipped type="unittest.case.SkipTest" message="unmigrated remainder from of massive profile test"><![CDATA[SkipTest: unmigrated remainder from of massive profile test
]]></skipped></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_profile_rdf" time="1.926"></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_profiles_by_interest" time="1.611"></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_public_profile_docs" time="1.560"><error type="exceptions.AttributeError" message="assert_called_once&#10;-------------------- &gt;&gt; begin captured logging &lt;&lt; --------------------&#10;eulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin&#10;pidservices.clients: DEBUG: Request: POST https://testpid.library.emory.edu/ark/ {'Content-Length': 3, 'Content-type': 'application/x-www-form-urlencoded', 'Accept': 'text/plain'} &lt;![BODY[{'domain': 'http://testpid.library.emory.edu/domains/34/', 'name': 'A very scholarly article', 'target_uri': u'http://example.com/publications/oe-zo-test:{%PID%}/'}]]&gt;&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /ark/ HTTP/1.1&quot; 201 None&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.012601 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.010573 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.015735 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.013312 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/objects/new HTTP/1.1&quot; 201 16&#10;eulfedora.api: DEBUG: POST objects/new=&gt;201: 0.555897 sec&#10;tracking.middleware: DEBUG: Created a new visitor: {'session_key': u'127.0.0.1:', 'ip_address': '127.0.0.1'}&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;DELETE /fedora/objects/oe-zo-test:pfz2r HTTP/1.1&quot; 200 24&#10;eulfedora.api: DEBUG: DELETE objects/oe-zo-test:pfz2r=&gt;200: 0.206291 sec&#10;--------------------- &gt;&gt; end captured logging &lt;&lt; ---------------------"><![CDATA[  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 67, in testPartExecutor
    yield
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 625, in run
    testMethod()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/Users/azotov/OpenEmory/openemory/accounts/tests.py", line 601, in test_public_profile_docs
    mockprofile.recent_articles.assert_called_once()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 721, in __getattr__
    raise AttributeError(name)
'assert_called_once\n-------------------- >> begin captured logging << --------------------\neulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin\npidservices.clients: DEBUG: Request: POST https://testpid.library.emory.edu/ark/ {\'Content-Length\': 3, \'Content-type\': \'application/x-www-form-urlencoded\', \'Accept\': \'text/plain\'} <![BODY[{\'domain\': \'http://testpid.library.emory.edu/domains/34/\', \'name\': \'A very scholarly article\', \'target_uri\': u\'http://example.com/publications/oe-zo-test:{%PID%}/\'}]]>]]&gt;<![CDATA[\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /ark/ HTTP/1.1" 201 None\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.012601 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.010573 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.015735 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.013312 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/objects/new HTTP/1.1" 201 16\neulfedora.api: DEBUG: POST objects/new=>201: 0.555897 sec\ntracking.middleware: DEBUG: Created a new visitor: {\'session_key\': u\'127.0.0.1:\', \'ip_address\': \'127.0.0.1\'}\nrequests.packages.urllib3.connectionpool: DEBUG: "DELETE /fedora/objects/oe-zo-test:pfz2r HTTP/1.1" 200 24\neulfedora.api: DEBUG: DELETE objects/oe-zo-test:pfz2r=>200: 0.206291 sec\n--------------------- >> end captured logging << ---------------------']]></error></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_public_profile_info" time="1.708"></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_statistics_processor" time="1.424"></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_tag_autocomplete" time="1.544"></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_tag_object_GET" time="2.181"></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_tag_object_PUT" time="1.659"></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_tag_profile_GET" time="1.423"></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_tag_profile_POST" time="1.493"></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_tag_profile_PUT" time="1.563"></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_tagged_items" time="1.634"></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_tags_in_sidebar" time="0.000"><skipped type="unittest.case.SkipTest" message="tags do not appear in contracted design"><![CDATA[SkipTest: tags do not appear in contracted design
]]></skipped></testcase><testcase classname="openemory.accounts.tests.AccountViewsTest" name="test_view_department" time="1.410"></testcase><testcase classname="openemory.accounts.tests.ArticlesByTagTest" name="test_articles_by_tag" time="0.028"></testcase><testcase classname="openemory.accounts.tests.ArticlesByTagTest" name="test_pids_by_tag" time="0.025"></testcase><testcase classname="openemory.accounts.tests.EsdPersonTest" name="test_first_name" time="0.197"></testcase><testcase classname="openemory.accounts.tests.EsdPersonTest" name="test_index_data" time="0.205"></testcase><testcase classname="openemory.accounts.tests.FacultyOrLocalAdminBackendTest" name="test_authenticate_esd_faculty" time="0.216"></testcase><testcase classname="openemory.accounts.tests.FacultyOrLocalAdminBackendTest" name="test_authenticate_local" time="0.207"></testcase><testcase classname="openemory.accounts.tests.LoginRequiredTest" name="test_anonymous" time="0.164"></testcase><testcase classname="openemory.accounts.tests.LoginRequiredTest" name="test_logged_in" time="0.133"></testcase><testcase classname="openemory.accounts.tests.LoginRequiredTest" name="test_wraps" time="0.131"></testcase><testcase classname="openemory.accounts.tests.PermissionRequiredTest" name="test_anonymous" time="0.169"></testcase><testcase classname="openemory.accounts.tests.PermissionRequiredTest" name="test_logged_in_allowed" time="0.132"></testcase><testcase classname="openemory.accounts.tests.PermissionRequiredTest" name="test_logged_in_notallowed" time="0.173"></testcase><testcase classname="openemory.accounts.tests.PermissionRequiredTest" name="test_wraps" time="0.136"></testcase><testcase classname="openemory.accounts.tests.ResarchersByInterestTestCase" name="test_researchers_by_interest" time="0.046"></testcase><testcase classname="openemory.accounts.tests.TagsTemplateFilterTest" name="test_anonymous" time="0.133"></testcase><testcase classname="openemory.accounts.tests.TagsTemplateFilterTest" name="test_bookmark" time="0.154"></testcase><testcase classname="openemory.accounts.tests.TagsTemplateFilterTest" name="test_no_bookmark" time="0.137"></testcase><testcase classname="openemory.accounts.tests.TagsTemplateFilterTest" name="test_no_pid" time="0.136"></testcase><testcase classname="openemory.accounts.tests.UserProfileTest" name="test_esd_data" time="0.211"></testcase><testcase classname="openemory.accounts.tests.UserProfileTest" name="test_find_articles" time="0.198"></testcase><testcase classname="openemory.accounts.tests.UserProfileTest" name="test_has_profile_page" time="0.219"></testcase><testcase classname="openemory.accounts.tests.UserProfileTest" name="test_recent_articles" time="0.210"><error type="exceptions.AttributeError" message="assert_called_once"><![CDATA[Traceback (most recent call last):
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 67, in testPartExecutor
    yield
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 625, in run
    testMethod()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/Users/azotov/OpenEmory/openemory/accounts/tests.py", line 2092, in test_recent_articles
    self.mocksolr.query.execute.assert_called_once()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 721, in __getattr__
    raise AttributeError(name)
AttributeError: assert_called_once
]]></error></testcase><testcase classname="openemory.accounts.tests.UserProfileTest" name="test_suppress_esd_data" time="0.214"></testcase><testcase classname="openemory.accounts.tests.UserProfileTest" name="test_unpublished_articles" time="0.201"><error type="exceptions.AttributeError" message="assert_called_once"><![CDATA[Traceback (most recent call last):
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 67, in testPartExecutor
    yield
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 625, in run
    testMethod()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/Users/azotov/OpenEmory/openemory/accounts/tests.py", line 2099, in test_unpublished_articles
    self.mocksolr.query.execute.assert_called_once()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 721, in __getattr__
    raise AttributeError(name)
AttributeError: assert_called_once
]]></error></testcase><testcase classname="openemory.common.tests.DigitalObjectTests" name="test_absolutize_url" time="0.002"></testcase><testcase classname="openemory.common.tests.DigitalObjectTests" name="test_get_default_pid" time="0.004"></testcase><testcase classname="openemory.common.tests.DigitalObjectTests" name="test_noid" time="0.000"></testcase><testcase classname="openemory.common.tests.RomeoTests" name="test_search_journal_issn_no_publisher" time="0.003"><error type="exceptions.AttributeError" message="assert_called_once"><![CDATA[Traceback (most recent call last):
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 67, in testPartExecutor
    yield
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 625, in run
    testMethod()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/Users/azotov/OpenEmory/openemory/common/tests.py", line 228, in test_search_journal_issn_no_publisher
    mock_urlopen.assert_called_once()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 721, in __getattr__
    raise AttributeError(name)
AttributeError: assert_called_once
]]></error></testcase><testcase classname="openemory.common.tests.RomeoTests" name="test_search_journal_issn_with_journal" time="0.003"><error type="exceptions.AttributeError" message="assert_called_once"><![CDATA[Traceback (most recent call last):
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 67, in testPartExecutor
    yield
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 625, in run
    testMethod()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/Users/azotov/OpenEmory/openemory/common/tests.py", line 203, in test_search_journal_issn_with_journal
    mock_urlopen.assert_called_once()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 721, in __getattr__
    raise AttributeError(name)
AttributeError: assert_called_once
]]></error></testcase><testcase classname="openemory.common.tests.RomeoTests" name="test_search_journal_title_multiple_example" time="0.002"><error type="exceptions.AttributeError" message="assert_called_once"><![CDATA[Traceback (most recent call last):
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 67, in testPartExecutor
    yield
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 625, in run
    testMethod()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/Users/azotov/OpenEmory/openemory/common/tests.py", line 155, in test_search_journal_title_multiple_example
    mock_urlopen.assert_called_once()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 721, in __getattr__
    raise AttributeError(name)
AttributeError: assert_called_once
]]></error></testcase><testcase classname="openemory.common.tests.RomeoTests" name="test_search_journal_title_no_match" time="0.002"><error type="exceptions.AttributeError" message="assert_called_once"><![CDATA[Traceback (most recent call last):
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 67, in testPartExecutor
    yield
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 625, in run
    testMethod()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/Users/azotov/OpenEmory/openemory/common/tests.py", line 248, in test_search_journal_title_no_match
    mock_urlopen.assert_called_once()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 721, in __getattr__
    raise AttributeError(name)
AttributeError: assert_called_once
]]></error></testcase><testcase classname="openemory.common.tests.RomeoTests" name="test_search_journal_title_single_example" time="0.002"><error type="exceptions.AttributeError" message="assert_called_once"><![CDATA[Traceback (most recent call last):
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 67, in testPartExecutor
    yield
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 625, in run
    testMethod()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/Users/azotov/OpenEmory/openemory/common/tests.py", line 180, in test_search_journal_title_single_example
    mock_urlopen.assert_called_once()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 721, in __getattr__
    raise AttributeError(name)
AttributeError: assert_called_once
]]></error></testcase><testcase classname="openemory.common.tests.RomeoTests" name="test_search_publisher_id_example" time="0.002"><error type="exceptions.AttributeError" message="assert_called_once"><![CDATA[Traceback (most recent call last):
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 67, in testPartExecutor
    yield
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 625, in run
    testMethod()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/Users/azotov/OpenEmory/openemory/common/tests.py", line 134, in test_search_publisher_id_example
    mock_urlopen.assert_called_once()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 721, in __getattr__
    raise AttributeError(name)
AttributeError: assert_called_once
]]></error></testcase><testcase classname="openemory.common.tests.RomeoTests" name="test_search_publisher_name_example" time="0.002"><error type="exceptions.AttributeError" message="assert_called_once"><![CDATA[Traceback (most recent call last):
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 67, in testPartExecutor
    yield
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 625, in run
    testMethod()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/Users/azotov/OpenEmory/openemory/common/tests.py", line 94, in test_search_publisher_name_example
    mock_urlopen.assert_called_once()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 721, in __getattr__
    raise AttributeError(name)
AttributeError: assert_called_once
]]></error></testcase><testcase classname="openemory.publication.tests.ArticleModsForm" name="test__license_desc" time="2.964"></testcase><testcase classname="openemory.publication.tests.ArticleModsTest" name="test_access_fields" time="0.002"></testcase><testcase classname="openemory.publication.tests.ArticleModsTest" name="test_author_name" time="0.002"></testcase><testcase classname="openemory.publication.tests.ArticleModsTest" name="test_author_note" time="0.000"></testcase><testcase classname="openemory.publication.tests.ArticleModsTest" name="test_calculate_embargo_end" time="0.008"></testcase><testcase classname="openemory.publication.tests.ArticleModsTest" name="test_create_mods_from_scratch" time="0.006"></testcase><testcase classname="openemory.publication.tests.ArticleModsTest" name="test_embargo" time="0.001"></testcase><testcase classname="openemory.publication.tests.ArticleModsTest" name="test_final_version" time="0.001"></testcase><testcase classname="openemory.publication.tests.ArticleModsTest" name="test_funding_group" time="0.002"></testcase><testcase classname="openemory.publication.tests.ArticleModsTest" name="test_keyword" time="0.000"></testcase><testcase classname="openemory.publication.tests.ArticleModsTest" name="test_publication_date" time="0.001"></testcase><testcase classname="openemory.publication.tests.ArticleModsTest" name="test_relateditem_isempty" time="0.004"></testcase><testcase classname="openemory.publication.tests.ArticleModsTest" name="test_researchfield" time="0.001"></testcase><testcase classname="openemory.publication.tests.ArticleModsTest" name="test_supplementalMaterial" time="0.001"></testcase><testcase classname="openemory.publication.tests.ArticlePremisTest" name="test_init_object" time="0.001"></testcase><testcase classname="openemory.publication.tests.ArticlePremisTest" name="test_premis_event" time="0.003"><error type="exceptions.Exception" message="No XSD schema is defined, cannot validate document&#10;-------------------- &gt;&gt; begin captured logging &lt;&lt; --------------------&#10;py.warnings: WARNING: /Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/eulxml/xmlmap/core.py:98: UserWarning: Loading schema http://www.loc.gov/standards/premis/v2/premis-v2-1.xsd without a web proxy may introduce significant network resource usage as well as instability if that server becomes inaccessible. The HTTP_PROXY environment variable is required for loading schemas.  Schema validation will be disabled.&#10;  warnings.warn(message, UserWarning)&#10;&#10;--------------------- &gt;&gt; end captured logging &lt;&lt; ---------------------"><![CDATA[  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 67, in testPartExecutor
    yield
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 625, in run
    testMethod()
  File "/Users/azotov/OpenEmory/openemory/publication/tests.py", line 3661, in test_premis_event
    self.assertTrue(pr.schema_valid())
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/eulxml/xmlmap/core.py", line 549, in schema_valid
    raise Exception('No XSD schema is defined, cannot validate document')
'No XSD schema is defined, cannot validate document\n-------------------- >> begin captured logging << --------------------\npy.warnings: WARNING: /Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/eulxml/xmlmap/core.py:98: UserWarning: Loading schema http://www.loc.gov/standards/premis/v2/premis-v2-1.xsd without a web proxy may introduce significant network resource usage as well as instability if that server becomes inaccessible. The HTTP_PROXY environment variable is required for loading schemas.  Schema validation will be disabled.\n  warnings.warn(message, UserWarning)\n\n--------------------- >> end captured logging << ---------------------']]></error></testcase><testcase classname="openemory.publication.tests.ArticlePremisTest" name="test_review_event" time="0.001"></testcase><testcase classname="openemory.publication.tests.ArticleTest" name="test__mods_to_dc" time="2.491"></testcase><testcase classname="openemory.publication.tests.ArticleTest" name="test__prep_dc_for_oai" time="2.380"></testcase><testcase classname="openemory.publication.tests.ArticleTest" name="test_as_rdf" time="2.539"></testcase><testcase classname="openemory.publication.tests.ArticleTest" name="test_as_symp" time="2.527"></testcase><testcase classname="openemory.publication.tests.ArticleTest" name="test_embargo_end_date" time="2.299"></testcase><testcase classname="openemory.publication.tests.ArticleTest" name="test_from_symp" time="6.586"></testcase><testcase classname="openemory.publication.tests.ArticleTest" name="test_index_data" time="3.852"><system-out><![CDATA[Do we get here 3???
Do we get here 5???
Do we get here 6???
Do we get here 8???
<p:premis xmlns:p="info:lc/xmlns/premis-v2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.1"><p:object xsi:type="p:representation"><p:objectIdentifier><p:objectIdentifierType>pid</p:objectIdentifierType><p:objectIdentifierValue>oe-zo-test:pfzvx</p:objectIdentifierValue></p:objectIdentifier></p:object><p:event><p:eventIdentifier><p:eventIdentifierType>local</p:eventIdentifierType><p:eventIdentifierValue>oe-zo-test:pfzvx.ev01</p:eventIdentifierValue></p:eventIdentifier><p:eventType>review</p:eventType><p:eventDateTime>2006-06-06T00:00:00.001</p:eventDateTime><p:eventDetail>reviewed by Ann Admynn</p:eventDetail><p:linkingAgentIdentifier><p:linkingAgentIdentifierType>netid</p:linkingAgentIdentifierType><p:linkingAgentIdentifierValue>aadmyn</p:linkingAgentIdentifierValue></p:linkingAgentIdentifier></p:event></p:premis>
Do we get here 7???
Do we get here 3???
Do we get here 5???
Do we get here 8???
<mods:mods xmlns:mods="http://www.loc.gov/mods/v3"><mods:titleInfo><mods:title>A very scholarly article</mods:title></mods:titleInfo><mods:identifier type="uri">http://testpid.library.emory.edu/ark:/25593/pfzvx</mods:identifier><mods:identifier type="ark">ark:/25593/pfzvx</mods:identifier><mods:originInfo><mods:dateOther type="embargoedUntil" encoding="w3cdtf">2015-08-22</mods:dateOther></mods:originInfo></mods:mods>
Do we get here 7???
]]></system-out></testcase><testcase classname="openemory.publication.tests.ArticleTest" name="test_is_embargoed" time="2.275"></testcase><testcase classname="openemory.publication.tests.ArticleTest" name="test_number_of_pages" time="2.410"></testcase><testcase classname="openemory.publication.tests.ArticleTest" name="test_pdf_cover" time="3.031"></testcase><testcase classname="openemory.publication.tests.ArticleTest" name="test_save" time="2.195"></testcase><testcase classname="openemory.publication.tests.ArticleTest" name="test_year_quarter" time="2.278"></testcase><testcase classname="openemory.publication.tests.AuthorNameFormTest" name="test_clean" time="0.001"></testcase><testcase classname="openemory.publication.tests.CodeListTest" name="test_access_fields" time="0.001"></testcase><testcase classname="openemory.publication.tests.LanguageCodeChoices" name="test_language_choices" time="0.002"></testcase><testcase classname="openemory.publication.tests.LanguageCodeChoices" name="test_language_codes" time="0.001"><error type="exceptions.AttributeError" message="assert_called_once"><![CDATA[Traceback (most recent call last):
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 67, in testPartExecutor
    yield
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 625, in run
    testMethod()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/Users/azotov/OpenEmory/openemory/publication/tests.py", line 3499, in test_language_codes
    mocklangcodes.assert_called_once()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 721, in __getattr__
    raise AttributeError(name)
AttributeError: assert_called_once
]]></error></testcase><testcase classname="openemory.publication.tests.LicenseChoices" name="test_license_choices" time="0.016"></testcase><testcase classname="openemory.publication.tests.NlmArticleTest" name="test_as_article_mods" time="0.247"></testcase><testcase classname="openemory.publication.tests.NlmArticleTest" name="test_basic_fields" time="0.137"></testcase><testcase classname="openemory.publication.tests.NlmArticleTest" name="test_fulltext_available" time="0.139"></testcase><testcase classname="openemory.publication.tests.NlmArticleTest" name="test_identifiable_authors" time="0.176"></testcase><testcase classname="openemory.publication.tests.NlmArticleTest" name="test_nlm_pubdate" time="0.139"></testcase><testcase classname="openemory.publication.tests.NlmLicenseTest" name="test_basic_fields" time="0.002"></testcase><testcase classname="openemory.publication.tests.NlmLicenseTest" name="test_creative_commons" time="0.002"></testcase><testcase classname="openemory.publication.tests.NlmLicenseTest" name="test_html" time="0.003"></testcase><testcase classname="openemory.publication.tests.NlmLicenseTest" name="test_text" time="0.002"></testcase><testcase classname="openemory.publication.tests.PdfToTextTest" name="test_file" time="0.982"></testcase><testcase classname="openemory.publication.tests.PdfToTextTest" name="test_object_datastream" time="1.173"></testcase><testcase classname="openemory.publication.tests.PdfToTextTest" name="test_unicode" time="1.061"></testcase><testcase classname="openemory.publication.tests.PdfToTextTest" name="test_whitespace" time="2.228"></testcase><testcase classname="openemory.publication.tests.PublicationViewsTest" name="test__parse_name" time="1.413"></testcase><testcase classname="openemory.publication.tests.PublicationViewsTest" name="test_audit_trail" time="1.664"></testcase><testcase classname="openemory.publication.tests.PublicationViewsTest" name="test_author_agreement" time="2.587"><system-out><![CDATA[Do we get here2???
Do we get here 5???
Do we get here 9???
True
Do we get here 7???
]]></system-out></testcase><testcase classname="openemory.publication.tests.PublicationViewsTest" name="test_browse" time="1.391"><error type="exceptions.AttributeError" message="assert_called_once&#10;-------------------- &gt;&gt; begin captured logging &lt;&lt; --------------------&#10;eulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin&#10;eulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin&#10;pidservices.clients: DEBUG: Request: POST https://testpid.library.emory.edu/ark/ {'Content-Length': 3, 'Content-type': 'application/x-www-form-urlencoded', 'Accept': 'text/plain'} &lt;![BODY[{'domain': 'http://testpid.library.emory.edu/domains/34/', 'name': 'A very scholarly article', 'target_uri': u'http://example.com/publications/oe-zo-test:{%PID%}/'}]]&gt;&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /ark/ HTTP/1.1&quot; 201 None&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.014724 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.015174 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.016189 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.011681 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/objects/new HTTP/1.1&quot; 201 16&#10;eulfedora.api: DEBUG: POST objects/new=&gt;201: 0.476188 sec&#10;tracking.middleware: DEBUG: Created a new visitor: {'session_key': u'127.0.0.1:', 'ip_address': '127.0.0.1'}&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;DELETE /fedora/objects/oe-zo-test:pg0hq HTTP/1.1&quot; 200 24&#10;eulfedora.api: DEBUG: DELETE objects/oe-zo-test:pg0hq=&gt;200: 0.216357 sec&#10;--------------------- &gt;&gt; end captured logging &lt;&lt; ---------------------"><![CDATA[  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 67, in testPartExecutor
    yield
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 625, in run
    testMethod()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/Users/azotov/OpenEmory/openemory/publication/tests.py", line 2946, in test_browse
    mocksolr.execute.assert_called_once()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 721, in __getattr__
    raise AttributeError(name)
'assert_called_once\n-------------------- >> begin captured logging << --------------------\neulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin\neulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin\npidservices.clients: DEBUG: Request: POST https://testpid.library.emory.edu/ark/ {\'Content-Length\': 3, \'Content-type\': \'application/x-www-form-urlencoded\', \'Accept\': \'text/plain\'} <![BODY[{\'domain\': \'http://testpid.library.emory.edu/domains/34/\', \'name\': \'A very scholarly article\', \'target_uri\': u\'http://example.com/publications/oe-zo-test:{%PID%}/\'}]]>]]&gt;<![CDATA[\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /ark/ HTTP/1.1" 201 None\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.014724 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.015174 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.016189 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.011681 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/objects/new HTTP/1.1" 201 16\neulfedora.api: DEBUG: POST objects/new=>201: 0.476188 sec\ntracking.middleware: DEBUG: Created a new visitor: {\'session_key\': u\'127.0.0.1:\', \'ip_address\': \'127.0.0.1\'}\nrequests.packages.urllib3.connectionpool: DEBUG: "DELETE /fedora/objects/oe-zo-test:pg0hq HTTP/1.1" 200 24\neulfedora.api: DEBUG: DELETE objects/oe-zo-test:pg0hq=>200: 0.216357 sec\n--------------------- >> end captured logging << ---------------------']]></error></testcase><testcase classname="openemory.publication.tests.PublicationViewsTest" name="test_edit_metadata" time="21.962"><system-out><![CDATA[Do we get here 3???
Do we get here 5???
Do we get here 8???
<mods:mods xmlns:mods="http://www.loc.gov/mods/v3"><mods:titleInfo><mods:title>A very scholarly article</mods:title></mods:titleInfo><mods:abstract>An overly complicated description of a very scholarly article</mods:abstract><mods:relatedItem type="host"><mods:originInfo><mods:publisher>Big Deal Publications</mods:publisher></mods:originInfo></mods:relatedItem><mods:identifier type="uri">http://testpid.library.emory.edu/ark:/25593/pg0jv</mods:identifier><mods:identifier type="ark">ark:/25593/pg0jv</mods:identifier><mods:name type="personal"><mods:affiliation>Mars Polytechnic Institute</mods:affiliation><mods:namePart type="given">Martian</mods:namePart><mods:namePart type="family">Manhunter</mods:namePart><mods:role><mods:roleTerm type="text">author</mods:roleTerm></mods:role></mods:name></mods:mods>
Do we get here 7???
Do we get here 3???
Do we get here 5???
Do we get here 8???
<oai_dc:dc xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/oai_dc/ http://www.openarchives.org/OAI/2.0/oai_dc.xsd">
  <dc:title>A very scholarly article</dc:title>
  <dc:description>An overly complicated description of a very scholarly article</dc:description>
  <dc:type>text</dc:type>
  <dc:type>article</dc:type>
  <dc:identifier>oe-zo-test:pg0jv</dc:identifier>
  <dc:identifier>http://testpid.library.emory.edu/ark:/25593/pg0jv</dc:identifier>
  <dc:source>None Volume  Issue  Date None Pages -</dc:source>
<dc:contributor>Martian Manhunter</dc:contributor></oai_dc:dc>
Do we get here 7???
Do we get here 3???
Do we get here 5???
Do we get here 8???
<mods:mods xmlns:mods="http://www.loc.gov/mods/v3"><mods:titleInfo><mods:title>Capitalism and the Origins of the Humanitarian Sensibility</mods:title></mods:titleInfo><mods:identifier type="uri">http://testpid.library.emory.edu/ark:/25593/pg0jv</mods:identifier><mods:identifier type="ark">ark:/25593/pg0jv</mods:identifier><mods:name type="personal"><mods:role><mods:roleTerm type="text">author</mods:roleTerm></mods:role></mods:name></mods:mods>
Do we get here 7???
Do we get here 3???
Do we get here 5???
Do we get here 8???
<oai_dc:dc xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/oai_dc/ http://www.openarchives.org/OAI/2.0/oai_dc.xsd">
  <dc:title>Capitalism and the Origins of the Humanitarian Sensibility</dc:title>
  <dc:description>An overly complicated description of a very scholarly article</dc:description>
  <dc:type>text</dc:type>
  <dc:type>article</dc:type>
  <dc:identifier>oe-zo-test:pg0jv</dc:identifier>
  <dc:identifier>http://testpid.library.emory.edu/ark:/25593/pg0jv</dc:identifier>
  <dc:source>None Volume  Issue  Date None Pages -</dc:source>
<dc:contributor>None None</dc:contributor></oai_dc:dc>
Do we get here 7???
obj is saved
oe-zo-test:pg0jv
Do we get here 3???
Do we get here 5???
Do we get here 8???
<mods:mods xmlns:mods="http://www.loc.gov/mods/v3"><mods:titleInfo><mods:title>Capitalism and the Origins of the Humanitarian Sensibility</mods:title></mods:titleInfo><mods:identifier type="uri">http://testpid.library.emory.edu/ark:/25593/pg0jv</mods:identifier><mods:identifier type="ark">ark:/25593/pg0jv</mods:identifier><mods:name type="personal"><mods:role><mods:roleTerm type="text">author</mods:roleTerm></mods:role></mods:name><mods:relatedItem type="host"><mods:titleInfo><mods:title>The American Historical Review</mods:title></mods:titleInfo><mods:originInfo><mods:publisher>American Historical Association</mods:publisher></mods:originInfo></mods:relatedItem><mods:genre authority="local">Preprint: Prior to Peer Review</mods:genre><mods:originInfo><mods:dateIssued encoding="w3cdtf" keyDate="yes">2005-01</mods:dateIssued></mods:originInfo><mods:language><mods:languageTerm type="code" authority="iso639-2b">eng</mods:languageTerm><mods:languageTerm type="text">English</mods:languageTerm></mods:language><mods:subject authority="proquestresearchfield" ID="id0729"><mods:topic>Cinema</mods:topic></mods:subject></mods:mods>
Do we get here 7???
Do we get here 3???
Do we get here 5???
Do we get here 6???
Do we get here 8???
<?xml version="1.0" encoding="UTF-8"?>
<rdf:RDF
   xmlns:fedora-model="info:fedora/fedora-system:def/model#"
   xmlns:oai="http://www.openarchives.org/OAI/2.0/"
   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
>
  <rdf:Description rdf:about="info:fedora/oe-zo-test:pg0jv">
    <oai:itemID>oai:ark:/25593/pg0jv</oai:itemID>
    <fedora-model:hasModel rdf:resource="info:fedora/emory-control:PublishedArticle-1.0"/>
  </rdf:Description>
</rdf:RDF>

Do we get here 7???
Do we get here 3???
Do we get here 5???
Do we get here 8???
<oai_dc:dc xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/oai_dc/ http://www.openarchives.org/OAI/2.0/oai_dc.xsd">
  <dc:title>Capitalism and the Origins of the Humanitarian Sensibility</dc:title>
  <dc:description>An overly complicated description of a very scholarly article</dc:description>
  <dc:type>text</dc:type>
  <dc:type>Preprint: Prior to Peer Review: article</dc:type>
  <dc:identifier>oe-zo-test:pg0jv</dc:identifier>
  <dc:identifier>http://testpid.library.emory.edu/ark:/25593/pg0jv</dc:identifier>
  <dc:source>The American Historical Review Volume  Issue  Date 2005-01 Pages -</dc:source>
<dc:contributor>None None</dc:contributor><dc:language>English</dc:language><dc:subject>Cinema</dc:subject></oai_dc:dc>
Do we get here 7???
obj is saved
Do we get here 3???
Do we get here 5???
Do we get here 8???
<mods:mods xmlns:mods="http://www.loc.gov/mods/v3"><mods:titleInfo><mods:title>Capitalism and the Origins of the Humanitarian Sensibility</mods:title><mods:subTitle>a critical approach</mods:subTitle><mods:partNumber>The Beginning</mods:partNumber><mods:partName>Part 1</mods:partName></mods:titleInfo><mods:identifier type="uri">http://testpid.library.emory.edu/ark:/25593/pg0jv</mods:identifier><mods:identifier type="ark">ark:/25593/pg0jv</mods:identifier><mods:relatedItem type="host"><mods:titleInfo><mods:title>The American Historical Review</mods:title></mods:titleInfo><mods:originInfo><mods:publisher>American Historical Association</mods:publisher></mods:originInfo><mods:part><mods:detail type="volume"><mods:number>90</mods:number></mods:detail><mods:detail type="number"><mods:number>2</mods:number></mods:detail><mods:extent unit="pages"><mods:start>331</mods:start><mods:end>361</mods:end></mods:extent></mods:part></mods:relatedItem><mods:genre authority="local">Preprint: Prior to Peer Review</mods:genre><mods:originInfo><mods:dateIssued encoding="w3cdtf" keyDate="yes">2005-01</mods:dateIssued><mods:dateOther type="embargoedUntil" encoding="w3cdtf">2006-02-01</mods:dateOther></mods:originInfo><mods:language><mods:languageTerm type="code" authority="iso639-2b">eng</mods:languageTerm><mods:languageTerm type="text">English</mods:languageTerm></mods:language><mods:subject authority="proquestresearchfield" ID="id0900"><mods:topic>Cinema</mods:topic></mods:subject><mods:relatedItem type="otherVersion" displayLabel="Final Published Version"><mods:identifier type="uri" displayLabel="URL">http://example.com/</mods:identifier><mods:identifier type="doi" displayLabel="DOI">doi:10.34/test/valid/doi</mods:identifier></mods:relatedItem><mods:abstract>An unprecedented wave of humanitarian reform sentiment swept through the societies of Western Europe, England, and North America in the hundred years following 1750.  Etc.</mods:abstract><mods:name type="corporate"><mods:role><mods:roleTerm type="text">funder</mods:roleTerm></mods:role><mods:namePart>Mellon Foundation</mods:namePart></mods:name><mods:relatedItem xmlns:xlink="http://www.w3.org/1999/xlink" type="references" displayLabel="SupplementalMaterial" xlink:href="http://someurl.com"/><mods:subject authority="keywords"><mods:topic>morality of capitalism</mods:topic></mods:subject><mods:note type="author notes">This paper was first given at the American Historical Association conference in 1943</mods:note><mods:accessCondition type="restrictionOnAccess">Embargoed for 12-months</mods:accessCondition><mods:name type="personal" ID="testuser"><mods:role><mods:roleTerm type="text">author</mods:roleTerm></mods:role><mods:namePart type="family">Tester</mods:namePart><mods:namePart type="given">Sue</mods:namePart><mods:affiliation>Emory University</mods:affiliation></mods:name></mods:mods>
Do we get here 7???
Do we get here???
Do we get here 5???
Do we get here 9???
True
Do we get here 7???
Do we get here 3???
Do we get here 5???
Do we get here 8???
<oai_dc:dc xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/oai_dc/ http://www.openarchives.org/OAI/2.0/oai_dc.xsd">
  <dc:title>Capitalism and the Origins of the Humanitarian Sensibility: a critical approach</dc:title>
  <dc:description>An unprecedented wave of humanitarian reform sentiment swept through the societies of Western Europe, England, and North America in the hundred years following 1750.  Etc.</dc:description>
  <dc:type>text</dc:type>
  <dc:type>Preprint: Prior to Peer Review: article</dc:type>
  <dc:identifier>oe-zo-test:pg0jv</dc:identifier>
  <dc:identifier>http://testpid.library.emory.edu/ark:/25593/pg0jv</dc:identifier>
  <dc:source>The American Historical Review Volume 90 Issue 2 Date 2005-01 Pages 331-361</dc:source>
<dc:contributor>Sue Tester</dc:contributor><dc:language>English</dc:language><dc:subject>Cinema</dc:subject><dc:subject>morality of capitalism</dc:subject></oai_dc:dc>
Do we get here 7???
obj is saved
Do we get here 3???
Do we get here 5???
Do we get here 8???
<mods:mods xmlns:mods="http://www.loc.gov/mods/v3"><mods:titleInfo><mods:title>Capitalism and the Origins of the Humanitarian Sensibility</mods:title><mods:subTitle>a critical approach</mods:subTitle><mods:partNumber>The Beginning</mods:partNumber><mods:partName>Part 1</mods:partName></mods:titleInfo><mods:identifier type="uri">http://testpid.library.emory.edu/ark:/25593/pg0jv</mods:identifier><mods:identifier type="ark">ark:/25593/pg0jv</mods:identifier><mods:relatedItem type="host"><mods:titleInfo><mods:title>The American Historical Review</mods:title></mods:titleInfo><mods:originInfo><mods:publisher>American Historical Association</mods:publisher></mods:originInfo><mods:part><mods:detail type="volume"><mods:number>90</mods:number></mods:detail><mods:detail type="number"><mods:number>2</mods:number></mods:detail><mods:extent unit="pages"><mods:start>331</mods:start><mods:end>361</mods:end></mods:extent></mods:part></mods:relatedItem><mods:genre authority="local">Preprint: Prior to Peer Review</mods:genre><mods:originInfo><mods:dateIssued encoding="w3cdtf" keyDate="yes">2005-01</mods:dateIssued></mods:originInfo><mods:language><mods:languageTerm type="code" authority="iso639-2b">eng</mods:languageTerm><mods:languageTerm type="text">English</mods:languageTerm></mods:language><mods:subject authority="proquestresearchfield" ID="id0900"><mods:topic>Cinema</mods:topic></mods:subject><mods:relatedItem type="otherVersion" displayLabel="Final Published Version"><mods:identifier type="uri" displayLabel="URL">http://example.com/</mods:identifier><mods:identifier type="doi" displayLabel="DOI">doi:10.34/test/valid/doi</mods:identifier></mods:relatedItem><mods:abstract>I came from a Windows machine so I have unnecessary control characters</mods:abstract><mods:name type="corporate"><mods:role><mods:roleTerm type="text">funder</mods:roleTerm></mods:role><mods:namePart>Mellon Foundation</mods:namePart></mods:name><mods:relatedItem xmlns:xlink="http://www.w3.org/1999/xlink" type="references" displayLabel="SupplementalMaterial" xlink:href="http://someurl.com"/><mods:subject authority="keywords"><mods:topic>morality of capitalism</mods:topic></mods:subject><mods:note type="author notes">This paper was first given at the American Historical Association conference in 1943</mods:note><mods:name type="personal" ID="testuser"><mods:role><mods:roleTerm type="text">author</mods:roleTerm></mods:role><mods:namePart type="family">Tester</mods:namePart><mods:namePart type="given">Sue</mods:namePart><mods:affiliation>Emory University</mods:affiliation></mods:name></mods:mods>
Do we get here 7???
Do we get here 3???
Do we get here 5???
Do we get here 8???
<oai_dc:dc xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/oai_dc/ http://www.openarchives.org/OAI/2.0/oai_dc.xsd">
  <dc:title>Capitalism and the Origins of the Humanitarian Sensibility: a critical approach</dc:title>
  <dc:description>I came from a Windows machine so I have unnecessary control characters</dc:description>
  <dc:type>text</dc:type>
  <dc:type>Preprint: Prior to Peer Review: article</dc:type>
  <dc:identifier>oe-zo-test:pg0jv</dc:identifier>
  <dc:identifier>http://testpid.library.emory.edu/ark:/25593/pg0jv</dc:identifier>
  <dc:source>The American Historical Review Volume 90 Issue 2 Date 2005-01 Pages 331-361</dc:source>
<dc:contributor>Sue Tester</dc:contributor><dc:language>English</dc:language><dc:subject>Cinema</dc:subject><dc:subject>morality of capitalism</dc:subject></oai_dc:dc>
Do we get here 7???
obj is saved
Do we get here 3???
Do we get here 5???
Do we get here 6???
Do we get here 8???
<p:premis xmlns:p="info:lc/xmlns/premis-v2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.1"><p:object xsi:type="p:representation"><p:objectIdentifier><p:objectIdentifierType>pid</p:objectIdentifierType><p:objectIdentifierValue>oe-zo-test:pg0jv</p:objectIdentifierValue></p:objectIdentifier></p:object><p:event><p:eventIdentifier><p:eventIdentifierType>local</p:eventIdentifierType><p:eventIdentifierValue>oe-zo-test:pg0jv.ev001</p:eventIdentifierValue></p:eventIdentifier><p:eventType>review</p:eventType><p:eventDateTime>2015-08-21T15:53:57.575762</p:eventDateTime><p:eventDetail>Reviewed by </p:eventDetail><p:linkingAgentIdentifier><p:linkingAgentIdentifierType>netid</p:linkingAgentIdentifierType><p:linkingAgentIdentifierValue>testuser</p:linkingAgentIdentifierValue></p:linkingAgentIdentifier></p:event></p:premis>
Do we get here 7???
Do we get here 3???
Do we get here 5???
Do we get here 8???
<mods:mods xmlns:mods="http://www.loc.gov/mods/v3"><mods:titleInfo><mods:title>Capitalism and the Origins of the Humanitarian Sensibility</mods:title><mods:subTitle>a critical approach</mods:subTitle><mods:partNumber>The Beginning</mods:partNumber><mods:partName>Part 1</mods:partName></mods:titleInfo><mods:identifier type="uri">http://testpid.library.emory.edu/ark:/25593/pg0jv</mods:identifier><mods:identifier type="ark">ark:/25593/pg0jv</mods:identifier><mods:relatedItem type="host"><mods:titleInfo><mods:title>The American Historical Review</mods:title></mods:titleInfo><mods:originInfo><mods:publisher>American Historical Association</mods:publisher></mods:originInfo><mods:part><mods:detail type="volume"><mods:number>90</mods:number></mods:detail><mods:detail type="number"><mods:number>2</mods:number></mods:detail><mods:extent unit="pages"><mods:start>331</mods:start><mods:end>361</mods:end></mods:extent></mods:part></mods:relatedItem><mods:genre authority="local">Preprint: Prior to Peer Review</mods:genre><mods:originInfo><mods:dateIssued encoding="w3cdtf" keyDate="yes">2005-01</mods:dateIssued></mods:originInfo><mods:language><mods:languageTerm type="code" authority="iso639-2b">eng</mods:languageTerm><mods:languageTerm type="text">English</mods:languageTerm></mods:language><mods:subject authority="proquestresearchfield" ID="id0900"><mods:topic>Cinema</mods:topic></mods:subject><mods:relatedItem type="otherVersion" displayLabel="Final Published Version"><mods:identifier type="uri" displayLabel="URL">http://example.com/</mods:identifier><mods:identifier type="doi" displayLabel="DOI">doi:10.34/test/valid/doi</mods:identifier></mods:relatedItem><mods:abstract>I came from a Windows machine so I have unnecessary control characters</mods:abstract><mods:name type="corporate"><mods:role><mods:roleTerm type="text">funder</mods:roleTerm></mods:role><mods:namePart>Mellon Foundation</mods:namePart></mods:name><mods:relatedItem xmlns:xlink="http://www.w3.org/1999/xlink" type="references" displayLabel="SupplementalMaterial" xlink:href="http://someurl.com"/><mods:subject authority="keywords"><mods:topic>morality of capitalism</mods:topic></mods:subject><mods:note type="author notes">This paper was first given at the American Historical Association conference in 1943</mods:note><mods:name type="personal" ID="testuser"><mods:role><mods:roleTerm type="text">author</mods:roleTerm></mods:role><mods:namePart type="family">Tester</mods:namePart><mods:namePart type="given">Sue</mods:namePart><mods:affiliation>Emory University</mods:affiliation></mods:name><mods:accessCondition type="restrictions on access" displayLabel="copyrightStatusDeterminationDate">2015-01-15</mods:accessCondition></mods:mods>
Do we get here 7???
Do we get here 3???
Do we get here 5???
Do we get here 6???
Do we get here 8???
<?xml version="1.0" encoding="UTF-8"?>
<rdf:RDF
   xmlns:fedora-model="info:fedora/fedora-system:def/model#"
   xmlns:oai="http://www.openarchives.org/OAI/2.0/"
   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
   xmlns:symp="info:symplectic/symplectic-elements:def/model#"
>
  <rdf:Description rdf:about="info:fedora/oe-zo-test:pg0jv">
    <oai:itemID>oai:ark:/25593/pg0jv</oai:itemID>
    <fedora-model:hasModel rdf:resource="info:fedora/emory-control:PublishedArticle-1.0"/>
    <symp:hasPublicUrl rdf:resource="https://testpid.library.emory.edu/ark:/25593/pg0jv"/>
  </rdf:Description>
</rdf:RDF>

Do we get here 7???
obj is saved
obj is saved
Do we get here 3???
Do we get here 5???
Do we get here 6???
Do we get here 8???
<p:premis xmlns:p="info:lc/xmlns/premis-v2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.1"><p:object xsi:type="p:representation"><p:objectIdentifier><p:objectIdentifierType>pid</p:objectIdentifierType><p:objectIdentifierValue>oe-zo-test:pg0jv</p:objectIdentifierValue></p:objectIdentifier></p:object><p:event><p:eventIdentifier><p:eventIdentifierType>local</p:eventIdentifierType><p:eventIdentifierValue>oe-zo-test:pg0jv.ev001</p:eventIdentifierValue></p:eventIdentifier><p:eventType>review</p:eventType><p:eventDateTime>2015-08-21T15:53:57.575762</p:eventDateTime><p:eventDetail>Reviewed by </p:eventDetail><p:linkingAgentIdentifier><p:linkingAgentIdentifierType>netid</p:linkingAgentIdentifierType><p:linkingAgentIdentifierValue>testuser</p:linkingAgentIdentifierValue></p:linkingAgentIdentifier></p:event><p:event><p:eventIdentifier><p:eventIdentifierType>local</p:eventIdentifierType><p:eventIdentifierValue>oe-zo-test:pg0jv.ev002</p:eventIdentifierValue></p:eventIdentifier><p:eventType>withdraw</p:eventType><p:eventDateTime>2015-08-21T15:54:01.300345</p:eventDateTime><p:eventDetail>Withdrawn by : test reason abcd</p:eventDetail><p:linkingAgentIdentifier><p:linkingAgentIdentifierType>netid</p:linkingAgentIdentifierType><p:linkingAgentIdentifierValue>testuser</p:linkingAgentIdentifierValue></p:linkingAgentIdentifier></p:event></p:premis>
Do we get here 7???
obj is saved
Do we get here 3???
Do we get here 5???
Do we get here 6???
Do we get here 8???
<p:premis xmlns:p="info:lc/xmlns/premis-v2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.1"><p:object xsi:type="p:representation"><p:objectIdentifier><p:objectIdentifierType>pid</p:objectIdentifierType><p:objectIdentifierValue>oe-zo-test:pg0jv</p:objectIdentifierValue></p:objectIdentifier></p:object><p:event><p:eventIdentifier><p:eventIdentifierType>local</p:eventIdentifierType><p:eventIdentifierValue>oe-zo-test:pg0jv.ev001</p:eventIdentifierValue></p:eventIdentifier><p:eventType>review</p:eventType><p:eventDateTime>2015-08-21T15:53:57.575762</p:eventDateTime><p:eventDetail>Reviewed by </p:eventDetail><p:linkingAgentIdentifier><p:linkingAgentIdentifierType>netid</p:linkingAgentIdentifierType><p:linkingAgentIdentifierValue>testuser</p:linkingAgentIdentifierValue></p:linkingAgentIdentifier></p:event><p:event><p:eventIdentifier><p:eventIdentifierType>local</p:eventIdentifierType><p:eventIdentifierValue>oe-zo-test:pg0jv.ev002</p:eventIdentifierValue></p:eventIdentifier><p:eventType>withdraw</p:eventType><p:eventDateTime>2015-08-21T15:54:01.300345</p:eventDateTime><p:eventDetail>Withdrawn by : test reason abcd</p:eventDetail><p:linkingAgentIdentifier><p:linkingAgentIdentifierType>netid</p:linkingAgentIdentifierType><p:linkingAgentIdentifierValue>testuser</p:linkingAgentIdentifierValue></p:linkingAgentIdentifier></p:event><p:event><p:eventIdentifier><p:eventIdentifierType>local</p:eventIdentifierType><p:eventIdentifierValue>oe-zo-test:pg0jv.ev003</p:eventIdentifierValue></p:eventIdentifier><p:eventType>reinstate</p:eventType><p:eventDateTime>2015-08-21T15:54:03.415411</p:eventDateTime><p:eventDetail>Reinstated (from withdrawal) by : test reason efgh</p:eventDetail><p:linkingAgentIdentifier><p:linkingAgentIdentifierType>netid</p:linkingAgentIdentifierType><p:linkingAgentIdentifierValue>testuser</p:linkingAgentIdentifierValue></p:linkingAgentIdentifier></p:event></p:premis>
Do we get here 7???
obj is saved
Do we get here 3???
Do we get here 5???
Do we get here 6???
Do we get here 8???
<p:premis xmlns:p="info:lc/xmlns/premis-v2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.1"><p:object xsi:type="p:representation"><p:objectIdentifier><p:objectIdentifierType>pid</p:objectIdentifierType><p:objectIdentifierValue>oe-zo-test:pg0jv</p:objectIdentifierValue></p:objectIdentifier></p:object><p:event><p:eventIdentifier><p:eventIdentifierType>local</p:eventIdentifierType><p:eventIdentifierValue>oe-zo-test:pg0jv.ev001</p:eventIdentifierValue></p:eventIdentifier><p:eventType>review</p:eventType><p:eventDateTime>2015-08-21T15:53:57.575762</p:eventDateTime><p:eventDetail>Reviewed by </p:eventDetail><p:linkingAgentIdentifier><p:linkingAgentIdentifierType>netid</p:linkingAgentIdentifierType><p:linkingAgentIdentifierValue>testuser</p:linkingAgentIdentifierValue></p:linkingAgentIdentifier></p:event><p:event><p:eventIdentifier><p:eventIdentifierType>local</p:eventIdentifierType><p:eventIdentifierValue>oe-zo-test:pg0jv.ev002</p:eventIdentifierValue></p:eventIdentifier><p:eventType>withdraw</p:eventType><p:eventDateTime>2015-08-21T15:54:01.300345</p:eventDateTime><p:eventDetail>Withdrawn by : test reason abcd</p:eventDetail><p:linkingAgentIdentifier><p:linkingAgentIdentifierType>netid</p:linkingAgentIdentifierType><p:linkingAgentIdentifierValue>testuser</p:linkingAgentIdentifierValue></p:linkingAgentIdentifier></p:event><p:event><p:eventIdentifier><p:eventIdentifierType>local</p:eventIdentifierType><p:eventIdentifierValue>oe-zo-test:pg0jv.ev003</p:eventIdentifierValue></p:eventIdentifier><p:eventType>reinstate</p:eventType><p:eventDateTime>2015-08-21T15:54:03.415411</p:eventDateTime><p:eventDetail>Reinstated (from withdrawal) by : test reason efgh</p:eventDetail><p:linkingAgentIdentifier><p:linkingAgentIdentifierType>netid</p:linkingAgentIdentifierType><p:linkingAgentIdentifierValue>testuser</p:linkingAgentIdentifierValue></p:linkingAgentIdentifier></p:event><p:event><p:eventIdentifier><p:eventIdentifierType>local</p:eventIdentifierType><p:eventIdentifierValue>oe-zo-test:pg0jv.ev004</p:eventIdentifierValue></p:eventIdentifier><p:eventType>withdraw</p:eventType><p:eventDateTime>2015-08-21T15:54:04.496854</p:eventDateTime><p:eventDetail>Withdrawn by : test reason ijkl</p:eventDetail><p:linkingAgentIdentifier><p:linkingAgentIdentifierType>netid</p:linkingAgentIdentifierType><p:linkingAgentIdentifierValue>testuser</p:linkingAgentIdentifierValue></p:linkingAgentIdentifier></p:event></p:premis>
Do we get here 7???
obj is saved
Do we get here 3???
Do we get here 5???
Do we get here 6???
Do we get here 8???
<p:premis xmlns:p="info:lc/xmlns/premis-v2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.1"><p:object xsi:type="p:representation"><p:objectIdentifier><p:objectIdentifierType>pid</p:objectIdentifierType><p:objectIdentifierValue>oe-zo-test:pg0jv</p:objectIdentifierValue></p:objectIdentifier></p:object><p:event><p:eventIdentifier><p:eventIdentifierType>local</p:eventIdentifierType><p:eventIdentifierValue>oe-zo-test:pg0jv.ev001</p:eventIdentifierValue></p:eventIdentifier><p:eventType>review</p:eventType><p:eventDateTime>2015-08-21T15:53:57.575762</p:eventDateTime><p:eventDetail>Reviewed by </p:eventDetail><p:linkingAgentIdentifier><p:linkingAgentIdentifierType>netid</p:linkingAgentIdentifierType><p:linkingAgentIdentifierValue>testuser</p:linkingAgentIdentifierValue></p:linkingAgentIdentifier></p:event><p:event><p:eventIdentifier><p:eventIdentifierType>local</p:eventIdentifierType><p:eventIdentifierValue>oe-zo-test:pg0jv.ev002</p:eventIdentifierValue></p:eventIdentifier><p:eventType>withdraw</p:eventType><p:eventDateTime>2015-08-21T15:54:01.300345</p:eventDateTime><p:eventDetail>Withdrawn by : test reason abcd</p:eventDetail><p:linkingAgentIdentifier><p:linkingAgentIdentifierType>netid</p:linkingAgentIdentifierType><p:linkingAgentIdentifierValue>testuser</p:linkingAgentIdentifierValue></p:linkingAgentIdentifier></p:event><p:event><p:eventIdentifier><p:eventIdentifierType>local</p:eventIdentifierType><p:eventIdentifierValue>oe-zo-test:pg0jv.ev003</p:eventIdentifierValue></p:eventIdentifier><p:eventType>reinstate</p:eventType><p:eventDateTime>2015-08-21T15:54:03.415411</p:eventDateTime><p:eventDetail>Reinstated (from withdrawal) by : test reason efgh</p:eventDetail><p:linkingAgentIdentifier><p:linkingAgentIdentifierType>netid</p:linkingAgentIdentifierType><p:linkingAgentIdentifierValue>testuser</p:linkingAgentIdentifierValue></p:linkingAgentIdentifier></p:event><p:event><p:eventIdentifier><p:eventIdentifierType>local</p:eventIdentifierType><p:eventIdentifierValue>oe-zo-test:pg0jv.ev004</p:eventIdentifierValue></p:eventIdentifier><p:eventType>withdraw</p:eventType><p:eventDateTime>2015-08-21T15:54:04.496854</p:eventDateTime><p:eventDetail>Withdrawn by : test reason ijkl</p:eventDetail><p:linkingAgentIdentifier><p:linkingAgentIdentifierType>netid</p:linkingAgentIdentifierType><p:linkingAgentIdentifierValue>testuser</p:linkingAgentIdentifierValue></p:linkingAgentIdentifier></p:event><p:event><p:eventIdentifier><p:eventIdentifierType>local</p:eventIdentifierType><p:eventIdentifierValue>oe-zo-test:pg0jv.ev005</p:eventIdentifierValue></p:eventIdentifier><p:eventType>reinstate</p:eventType><p:eventDateTime>2015-08-21T15:54:05.544287</p:eventDateTime><p:eventDetail>Reinstated (from withdrawal) by : test reason mnop</p:eventDetail><p:linkingAgentIdentifier><p:linkingAgentIdentifierType>netid</p:linkingAgentIdentifierType><p:linkingAgentIdentifierValue>testuser</p:linkingAgentIdentifierValue></p:linkingAgentIdentifier></p:event></p:premis>
Do we get here 7???
obj is saved
]]></system-out></testcase><testcase classname="openemory.publication.tests.PublicationViewsTest" name="test_ingest_from_harvestrecord" time="4.590"><system-out><![CDATA[Do we get here 3???
Do we get here 5???
Do we get here 8???
<oai_dc:dc xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/" xmlns:dc="http://purl.org/dc/elements/1.1/"><dc:title/>
  <dc:creator/>
  <dc:type>text</dc:type>
  <dc:type>article</dc:type>
  <dc:identifier>oe-zo-test:pg0m4</dc:identifier>
  <dc:identifier>http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2001/</dc:identifier>
  <dc:identifier>PMC2001</dc:identifier>
  <dc:identifier>http://testpid.library.emory.edu/ark:/25593/pg0m4</dc:identifier>
  <dc:source>None Volume  Issue  Date None Pages -</dc:source>
</oai_dc:dc>
Do we get here 7???
Do we get here 3???
Do we get here 5???
Do we get here 6???
Do we get here 8???
<p:premis xmlns:p="info:lc/xmlns/premis-v2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.1"><p:object xsi:type="p:representation"><p:objectIdentifier><p:objectIdentifierType>pid</p:objectIdentifierType><p:objectIdentifierValue>oe-zo-test:pg0m4</p:objectIdentifierValue></p:objectIdentifier></p:object><p:event><p:eventIdentifier><p:eventIdentifierType>local</p:eventIdentifierType><p:eventIdentifierValue>oe-zo-test:pg0m4.ev001</p:eventIdentifierValue></p:eventIdentifier><p:eventType>harvest</p:eventType><p:eventDateTime>2015-08-21T15:54:09.840489</p:eventDateTime><p:eventDetail>Harvested 2001 from PubMed Central by </p:eventDetail><p:linkingAgentIdentifier><p:linkingAgentIdentifierType>netid</p:linkingAgentIdentifierType><p:linkingAgentIdentifierValue>testuser</p:linkingAgentIdentifierValue></p:linkingAgentIdentifier></p:event></p:premis>
Do we get here 7???
]]></system-out></testcase><testcase classname="openemory.publication.tests.PublicationViewsTest" name="test_ingest_upload" time="7.509"><system-out><![CDATA[Do we get here 3???
Do we get here 5???
Do we get here 8???
<oai_dc:dc xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/" xmlns:dc="http://purl.org/dc/elements/1.1/"><dc:title>test.pdf</dc:title>
  <dc:contributor> </dc:contributor>
  <dc:type>text</dc:type>
  <dc:type>article</dc:type>
  <dc:format>application/pdf</dc:format>
  <dc:identifier>oe-zo-test:pg0pd</dc:identifier>
  <dc:identifier>http://testpid.library.emory.edu/ark:/25593/pg0pd</dc:identifier>
</oai_dc:dc>
Do we get here 7???
Do we get here 3???
Do we get here 5???
Do we get here 6???
Do we get here 8???
<p:premis xmlns:p="info:lc/xmlns/premis-v2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.1"><p:object xsi:type="p:representation"><p:objectIdentifier><p:objectIdentifierType>pid</p:objectIdentifierType><p:objectIdentifierValue>oe-zo-test:pg0pd</p:objectIdentifierValue></p:objectIdentifier></p:object><p:event><p:eventIdentifier><p:eventIdentifierType>local</p:eventIdentifierType><p:eventIdentifierValue>oe-zo-test:pg0pd.ev001</p:eventIdentifierValue></p:eventIdentifier><p:eventType>upload</p:eventType><p:eventDateTime>2015-08-21T15:54:14.899364</p:eventDateTime><p:eventDetail>Uploaded by  upon assent to deposit under OpenEmory v1.2.0-dev</p:eventDetail><p:linkingAgentIdentifier><p:linkingAgentIdentifierType>netid</p:linkingAgentIdentifierType><p:linkingAgentIdentifierValue>testuser</p:linkingAgentIdentifierValue></p:linkingAgentIdentifier></p:event></p:premis>
Do we get here 7???
Do we get here 3???
Do we get here 5???
Do we get here 8???
<oai_dc:dc xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/" xmlns:dc="http://purl.org/dc/elements/1.1/"><dc:title>test.pdf</dc:title>
  <dc:type>text</dc:type>
  <dc:type>article</dc:type>
  <dc:format>application/pdf</dc:format>
  <dc:identifier>oe-zo-test:pg0qj</dc:identifier>
  <dc:identifier>http://testpid.library.emory.edu/ark:/25593/pg0qj</dc:identifier>
</oai_dc:dc>
Do we get here 7???
Do we get here 3???
Do we get here 5???
Do we get here 6???
Do we get here 8???
<p:premis xmlns:p="info:lc/xmlns/premis-v2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.1"><p:object xsi:type="p:representation"><p:objectIdentifier><p:objectIdentifierType>pid</p:objectIdentifierType><p:objectIdentifierValue>oe-zo-test:pg0qj</p:objectIdentifierValue></p:objectIdentifier></p:object><p:event><p:eventIdentifier><p:eventIdentifierType>local</p:eventIdentifierType><p:eventIdentifierValue>oe-zo-test:pg0qj.ev001</p:eventIdentifierValue></p:eventIdentifier><p:eventType>upload</p:eventType><p:eventDateTime>2015-08-21T15:54:17.617693</p:eventDateTime><p:eventDetail>Mediated Deposit with Assist Authorization or CC or PD by  under OpenEmory v1.2.0-dev</p:eventDetail><p:linkingAgentIdentifier><p:linkingAgentIdentifierType>netid</p:linkingAgentIdentifierType><p:linkingAgentIdentifierValue>testuser</p:linkingAgentIdentifierValue></p:linkingAgentIdentifier></p:event></p:premis>
Do we get here 7???
]]></system-out></testcase><testcase classname="openemory.publication.tests.PublicationViewsTest" name="test_pdf" time="5.498"><system-out><![CDATA[Do we get here 3???
Do we get here 5???
Do we get here 8???
<mods:mods xmlns:mods="http://www.loc.gov/mods/v3"><mods:titleInfo><mods:title>A very scholarly article</mods:title></mods:titleInfo><mods:abstract>An overly complicated description of a very scholarly article</mods:abstract><mods:relatedItem type="host"><mods:originInfo><mods:publisher>Big Deal Publications</mods:publisher></mods:originInfo></mods:relatedItem><mods:identifier type="uri">http://testpid.library.emory.edu/ark:/25593/pg0rp</mods:identifier><mods:identifier type="ark">ark:/25593/pg0rp</mods:identifier><mods:originInfo><mods:dateOther type="embargoedUntil" encoding="w3cdtf">2015-08-22</mods:dateOther></mods:originInfo></mods:mods>
Do we get here 7???
]]></system-out></testcase><testcase classname="openemory.publication.tests.PublicationViewsTest" name="test_rels_ext" time="1.849"><system-out><![CDATA[Do we get here 3???
Do we get here 5???
Do we get here 6???
Do we get here 8???
<p:premis xmlns:p="info:lc/xmlns/premis-v2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.1"><p:object xsi:type="p:representation"><p:objectIdentifier><p:objectIdentifierType>pid</p:objectIdentifierType><p:objectIdentifierValue>oe-zo-test:pg0st</p:objectIdentifierValue></p:objectIdentifier></p:object><p:event><p:eventIdentifier><p:eventIdentifierType>local</p:eventIdentifierType><p:eventIdentifierValue>oe-zo-test:pg0st.ev001</p:eventIdentifierValue></p:eventIdentifier><p:eventType>harvest</p:eventType><p:eventDateTime>2015-08-21T15:54:25.595556</p:eventDateTime><p:eventDetail>Harvested pmc123 from PubMed Central by Joe User</p:eventDetail><p:linkingAgentIdentifier><p:linkingAgentIdentifierType>netid</p:linkingAgentIdentifierType><p:linkingAgentIdentifierValue>juser</p:linkingAgentIdentifierValue></p:linkingAgentIdentifier></p:event><p:event><p:eventIdentifier><p:eventIdentifierType>local</p:eventIdentifierType><p:eventIdentifierValue>oe-zo-test:pg0st.ev002</p:eventIdentifierValue></p:eventIdentifier><p:eventType>review</p:eventType><p:eventDateTime>2015-08-21T15:54:25.596434</p:eventDateTime><p:eventDetail>Reviewed by Joe User</p:eventDetail><p:linkingAgentIdentifier><p:linkingAgentIdentifierType>netid</p:linkingAgentIdentifierType><p:linkingAgentIdentifierValue>juser</p:linkingAgentIdentifierValue></p:linkingAgentIdentifier></p:event></p:premis>
Do we get here 7???
]]></system-out></testcase><testcase classname="openemory.publication.tests.PublicationViewsTest" name="test_review_list" time="1.747"><error type="exceptions.AttributeError" message="assert_called&#10;-------------------- &gt;&gt; begin captured logging &lt;&lt; --------------------&#10;eulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin&#10;eulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin&#10;pidservices.clients: DEBUG: Request: POST https://testpid.library.emory.edu/ark/ {'Content-Length': 3, 'Content-type': 'application/x-www-form-urlencoded', 'Accept': 'text/plain'} &lt;![BODY[{'domain': 'http://testpid.library.emory.edu/domains/34/', 'name': 'A very scholarly article', 'target_uri': u'http://example.com/publications/oe-zo-test:{%PID%}/'}]]&gt;&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /ark/ HTTP/1.1&quot; 201 None&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.014926 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.012109 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.012538 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.012595 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/objects/new HTTP/1.1&quot; 201 16&#10;eulfedora.api: DEBUG: POST objects/new=&gt;201: 0.449768 sec&#10;tracking.middleware: DEBUG: Created a new visitor: {'session_key': u'127.0.0.1:', 'ip_address': '127.0.0.1'}&#10;tracking.middleware: DEBUG: Using existing visitor for IP 127.0.0.1 / UA : 32&#10;tracking.middleware: DEBUG: Using existing visitor for IP 127.0.0.1 / UA : 32&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;DELETE /fedora/objects/oe-zo-test:pg0tz HTTP/1.1&quot; 200 24&#10;eulfedora.api: DEBUG: DELETE objects/oe-zo-test:pg0tz=&gt;200: 0.206371 sec&#10;--------------------- &gt;&gt; end captured logging &lt;&lt; ---------------------"><![CDATA[  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 67, in testPartExecutor
    yield
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 625, in run
    testMethod()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/Users/azotov/OpenEmory/openemory/publication/tests.py", line 2808, in test_review_list
    mocksolr.query.assert_called()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 721, in __getattr__
    raise AttributeError(name)
'assert_called\n-------------------- >> begin captured logging << --------------------\neulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin\neulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin\npidservices.clients: DEBUG: Request: POST https://testpid.library.emory.edu/ark/ {\'Content-Length\': 3, \'Content-type\': \'application/x-www-form-urlencoded\', \'Accept\': \'text/plain\'} <![BODY[{\'domain\': \'http://testpid.library.emory.edu/domains/34/\', \'name\': \'A very scholarly article\', \'target_uri\': u\'http://example.com/publications/oe-zo-test:{%PID%}/\'}]]>]]&gt;<![CDATA[\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /ark/ HTTP/1.1" 201 None\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.014926 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.012109 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.012538 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.012595 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/objects/new HTTP/1.1" 201 16\neulfedora.api: DEBUG: POST objects/new=>201: 0.449768 sec\ntracking.middleware: DEBUG: Created a new visitor: {\'session_key\': u\'127.0.0.1:\', \'ip_address\': \'127.0.0.1\'}\ntracking.middleware: DEBUG: Using existing visitor for IP 127.0.0.1 / UA : 32\ntracking.middleware: DEBUG: Using existing visitor for IP 127.0.0.1 / UA : 32\nrequests.packages.urllib3.connectionpool: DEBUG: "DELETE /fedora/objects/oe-zo-test:pg0tz HTTP/1.1" 200 24\neulfedora.api: DEBUG: DELETE objects/oe-zo-test:pg0tz=>200: 0.206371 sec\n--------------------- >> end captured logging << ---------------------']]></error></testcase><testcase classname="openemory.publication.tests.PublicationViewsTest" name="test_review_list_ajax" time="1.661"></testcase><testcase classname="openemory.publication.tests.PublicationViewsTest" name="test_search_faceting" time="1.413"></testcase><testcase classname="openemory.publication.tests.PublicationViewsTest" name="test_search_keyword" time="1.451"></testcase><testcase classname="openemory.publication.tests.PublicationViewsTest" name="test_search_person" time="1.666"></testcase><testcase classname="openemory.publication.tests.PublicationViewsTest" name="test_search_phrase" time="1.484"></testcase><testcase classname="openemory.publication.tests.PublicationViewsTest" name="test_search_within" time="3.230"><error type="exceptions.AttributeError" message="assert_called_once&#10;-------------------- &gt;&gt; begin captured logging &lt;&lt; --------------------&#10;eulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin&#10;eulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin&#10;pidservices.clients: DEBUG: Request: POST https://testpid.library.emory.edu/ark/ {'Content-Length': 3, 'Content-type': 'application/x-www-form-urlencoded', 'Accept': 'text/plain'} &lt;![BODY[{'domain': 'http://testpid.library.emory.edu/domains/34/', 'name': 'A very scholarly article', 'target_uri': u'http://example.com/publications/oe-zo-test:{%PID%}/'}]]&gt;&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /ark/ HTTP/1.1&quot; 201 None&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.073728 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.164113 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.012967 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 1.601411 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/objects/new HTTP/1.1&quot; 201 16&#10;eulfedora.api: DEBUG: POST objects/new=&gt;201: 0.394233 sec&#10;tracking.middleware: DEBUG: Created a new visitor: {'session_key': u'127.0.0.1:', 'ip_address': '127.0.0.1'}&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;DELETE /fedora/objects/oe-zo-test:pg11w HTTP/1.1&quot; 200 24&#10;eulfedora.api: DEBUG: DELETE objects/oe-zo-test:pg11w=&gt;200: 0.228046 sec&#10;--------------------- &gt;&gt; end captured logging &lt;&lt; ---------------------"><![CDATA[  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 67, in testPartExecutor
    yield
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 625, in run
    testMethod()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/Users/azotov/OpenEmory/openemory/publication/tests.py", line 2157, in test_search_within
    mocksolr.execute.assert_called_once()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 721, in __getattr__
    raise AttributeError(name)
'assert_called_once\n-------------------- >> begin captured logging << --------------------\neulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin\neulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin\npidservices.clients: DEBUG: Request: POST https://testpid.library.emory.edu/ark/ {\'Content-Length\': 3, \'Content-type\': \'application/x-www-form-urlencoded\', \'Accept\': \'text/plain\'} <![BODY[{\'domain\': \'http://testpid.library.emory.edu/domains/34/\', \'name\': \'A very scholarly article\', \'target_uri\': u\'http://example.com/publications/oe-zo-test:{%PID%}/\'}]]>]]&gt;<![CDATA[\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /ark/ HTTP/1.1" 201 None\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.073728 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.164113 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.012967 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 1.601411 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/objects/new HTTP/1.1" 201 16\neulfedora.api: DEBUG: POST objects/new=>201: 0.394233 sec\ntracking.middleware: DEBUG: Created a new visitor: {\'session_key\': u\'127.0.0.1:\', \'ip_address\': \'127.0.0.1\'}\nrequests.packages.urllib3.connectionpool: DEBUG: "DELETE /fedora/objects/oe-zo-test:pg11w HTTP/1.1" 200 24\neulfedora.api: DEBUG: DELETE objects/oe-zo-test:pg11w=>200: 0.228046 sec\n--------------------- >> end captured logging << ---------------------']]></error></testcase><testcase classname="openemory.publication.tests.PublicationViewsTest" name="test_statistics_processor" time="1.526"></testcase><testcase classname="openemory.publication.tests.PublicationViewsTest" name="test_suggest" time="1.772"><error type="exceptions.AttributeError" message="assert_called_once&#10;-------------------- &gt;&gt; begin captured logging &lt;&lt; --------------------&#10;eulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin&#10;eulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin&#10;pidservices.clients: DEBUG: Request: POST https://testpid.library.emory.edu/ark/ {'Content-Length': 3, 'Content-type': 'application/x-www-form-urlencoded', 'Accept': 'text/plain'} &lt;![BODY[{'domain': 'http://testpid.library.emory.edu/domains/34/', 'name': 'A very scholarly article', 'target_uri': u'http://example.com/publications/oe-zo-test:{%PID%}/'}]]&gt;&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /ark/ HTTP/1.1&quot; 201 None&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.015023 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.012304 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.014728 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.013934 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/objects/new HTTP/1.1&quot; 201 16&#10;eulfedora.api: DEBUG: POST objects/new=&gt;201: 0.648833 sec&#10;tracking.middleware: DEBUG: Created a new visitor: {'session_key': u'127.0.0.1:', 'ip_address': '127.0.0.1'}&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;DELETE /fedora/objects/oe-zo-test:pg135 HTTP/1.1&quot; 200 24&#10;eulfedora.api: DEBUG: DELETE objects/oe-zo-test:pg135=&gt;200: 0.332763 sec&#10;--------------------- &gt;&gt; end captured logging &lt;&lt; ---------------------"><![CDATA[  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 67, in testPartExecutor
    yield
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 625, in run
    testMethod()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/Users/azotov/OpenEmory/openemory/publication/tests.py", line 2192, in test_suggest
    mocksolr.query.assert_called_once()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 721, in __getattr__
    raise AttributeError(name)
'assert_called_once\n-------------------- >> begin captured logging << --------------------\neulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin\neulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin\npidservices.clients: DEBUG: Request: POST https://testpid.library.emory.edu/ark/ {\'Content-Length\': 3, \'Content-type\': \'application/x-www-form-urlencoded\', \'Accept\': \'text/plain\'} <![BODY[{\'domain\': \'http://testpid.library.emory.edu/domains/34/\', \'name\': \'A very scholarly article\', \'target_uri\': u\'http://example.com/publications/oe-zo-test:{%PID%}/\'}]]>]]&gt;<![CDATA[\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /ark/ HTTP/1.1" 201 None\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.015023 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.012304 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.014728 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.013934 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/objects/new HTTP/1.1" 201 16\neulfedora.api: DEBUG: POST objects/new=>201: 0.648833 sec\ntracking.middleware: DEBUG: Created a new visitor: {\'session_key\': u\'127.0.0.1:\', \'ip_address\': \'127.0.0.1\'}\nrequests.packages.urllib3.connectionpool: DEBUG: "DELETE /fedora/objects/oe-zo-test:pg135 HTTP/1.1" 200 24\neulfedora.api: DEBUG: DELETE objects/oe-zo-test:pg135=>200: 0.332763 sec\n--------------------- >> end captured logging << ---------------------']]></error></testcase><testcase classname="openemory.publication.tests.PublicationViewsTest" name="test_suggest_journal_publisher" time="1.834"></testcase><testcase classname="openemory.publication.tests.PublicationViewsTest" name="test_suggest_journal_title" time="1.602"></testcase><testcase classname="openemory.publication.tests.PublicationViewsTest" name="test_summary" time="1.412"><error type="exceptions.AttributeError" message="assert_called&#10;-------------------- &gt;&gt; begin captured logging &lt;&lt; --------------------&#10;eulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin&#10;eulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin&#10;pidservices.clients: DEBUG: Request: POST https://testpid.library.emory.edu/ark/ {'Content-Length': 3, 'Content-type': 'application/x-www-form-urlencoded', 'Accept': 'text/plain'} &lt;![BODY[{'domain': 'http://testpid.library.emory.edu/domains/34/', 'name': 'A very scholarly article', 'target_uri': u'http://example.com/publications/oe-zo-test:{%PID%}/'}]]&gt;&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /ark/ HTTP/1.1&quot; 201 None&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.014514 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.011723 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.011029 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/upload HTTP/1.1&quot; 202 16&#10;eulfedora.api: DEBUG: POST upload=&gt;202: 0.013114 sec&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;POST /fedora/objects/new HTTP/1.1&quot; 201 16&#10;eulfedora.api: DEBUG: POST objects/new=&gt;201: 0.448956 sec&#10;tracking.middleware: DEBUG: Created a new visitor: {'session_key': u'127.0.0.1:', 'ip_address': '127.0.0.1'}&#10;requests.packages.urllib3.connectionpool: DEBUG: &quot;DELETE /fedora/objects/oe-zo-test:pg16k HTTP/1.1&quot; 200 23&#10;eulfedora.api: DEBUG: DELETE objects/oe-zo-test:pg16k=&gt;200: 0.209537 sec&#10;--------------------- &gt;&gt; end captured logging &lt;&lt; ---------------------"><![CDATA[  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 67, in testPartExecutor
    yield
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 625, in run
    testMethod()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/Users/azotov/OpenEmory/openemory/publication/tests.py", line 2877, in test_summary
    mocksolr.field_limit.assert_called()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 721, in __getattr__
    raise AttributeError(name)
'assert_called\n-------------------- >> begin captured logging << --------------------\neulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin\neulfedora.server: DEBUG: Connecting to fedora at https://libfeddev1.library.emory.edu:8443/fedora/ as fedoraAdmin\npidservices.clients: DEBUG: Request: POST https://testpid.library.emory.edu/ark/ {\'Content-Length\': 3, \'Content-type\': \'application/x-www-form-urlencoded\', \'Accept\': \'text/plain\'} <![BODY[{\'domain\': \'http://testpid.library.emory.edu/domains/34/\', \'name\': \'A very scholarly article\', \'target_uri\': u\'http://example.com/publications/oe-zo-test:{%PID%}/\'}]]>]]&gt;<![CDATA[\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /ark/ HTTP/1.1" 201 None\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.014514 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.011723 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.011029 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/upload HTTP/1.1" 202 16\neulfedora.api: DEBUG: POST upload=>202: 0.013114 sec\nrequests.packages.urllib3.connectionpool: DEBUG: "POST /fedora/objects/new HTTP/1.1" 201 16\neulfedora.api: DEBUG: POST objects/new=>201: 0.448956 sec\ntracking.middleware: DEBUG: Created a new visitor: {\'session_key\': u\'127.0.0.1:\', \'ip_address\': \'127.0.0.1\'}\nrequests.packages.urllib3.connectionpool: DEBUG: "DELETE /fedora/objects/oe-zo-test:pg16k HTTP/1.1" 200 23\neulfedora.api: DEBUG: DELETE objects/oe-zo-test:pg16k=>200: 0.209537 sec\n--------------------- >> end captured logging << ---------------------']]></error></testcase><testcase classname="openemory.publication.tests.PublicationViewsTest" name="test_view_article" time="5.341"><system-out><![CDATA[Do we get here 3???
Do we get here 5???
Do we get here 6???
Do we get here 8???
<p:premis xmlns:p="info:lc/xmlns/premis-v2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.1"><p:object xsi:type="p:representation"><p:objectIdentifier><p:objectIdentifierType>pid</p:objectIdentifierType><p:objectIdentifierValue>oe-zo-test:pg17q</p:objectIdentifierValue></p:objectIdentifier></p:object><p:event><p:eventIdentifier><p:eventIdentifierType>local</p:eventIdentifierType><p:eventIdentifierValue>oe-zo-test:pg17q.ev001</p:eventIdentifierValue></p:eventIdentifier><p:eventType>harvest</p:eventType><p:eventDateTime>2015-08-21T15:54:48.267335</p:eventDateTime><p:eventDetail>Harvested pmc123 from PubMed Central by Joe User</p:eventDetail><p:linkingAgentIdentifier><p:linkingAgentIdentifierType>netid</p:linkingAgentIdentifierType><p:linkingAgentIdentifierValue>juser</p:linkingAgentIdentifierValue></p:linkingAgentIdentifier></p:event><p:event><p:eventIdentifier><p:eventIdentifierType>local</p:eventIdentifierType><p:eventIdentifierValue>oe-zo-test:pg17q.ev002</p:eventIdentifierValue></p:eventIdentifier><p:eventType>review</p:eventType><p:eventDateTime>2015-08-21T15:54:48.268346</p:eventDateTime><p:eventDetail>Reviewed by Joe User</p:eventDetail><p:linkingAgentIdentifier><p:linkingAgentIdentifierType>netid</p:linkingAgentIdentifierType><p:linkingAgentIdentifierValue>juser</p:linkingAgentIdentifierValue></p:linkingAgentIdentifier></p:event></p:premis>
Do we get here 7???
Do we get here 3???
Do we get here 5???
Do we get here 8???
<mods:mods xmlns:mods="http://www.loc.gov/mods/v3"><mods:titleInfo><mods:title>A very scholarly article</mods:title><mods:partNumber>Part 1</mods:partNumber><mods:partName>Where we are now</mods:partName></mods:titleInfo><mods:abstract>An overly complicated description of a very scholarly article</mods:abstract><mods:relatedItem type="host"><mods:originInfo><mods:publisher>Nature Publishing Group</mods:publisher></mods:originInfo><mods:titleInfo><mods:title>Nature</mods:title></mods:titleInfo><mods:part><mods:detail type="volume"><mods:number>92</mods:number></mods:detail><mods:detail type="number"><mods:number>3</mods:number></mods:detail><mods:extent unit="pages"><mods:start>362</mods:start><mods:end>376</mods:end></mods:extent></mods:part></mods:relatedItem><mods:identifier type="uri">http://testpid.library.emory.edu/ark:/25593/pg17q</mods:identifier><mods:identifier type="ark">ark:/25593/pg17q</mods:identifier><mods:name ID="mmouse" type="personal"><mods:affiliation>Emory University</mods:affiliation><mods:namePart type="family">Mouse</mods:namePart><mods:namePart type="given">Minnie</mods:namePart><mods:role><mods:roleTerm type="text">author</mods:roleTerm></mods:role></mods:name><mods:name type="personal"><mods:affiliation>GA Tech</mods:affiliation><mods:namePart type="given">Joe</mods:namePart><mods:namePart type="family">Science</mods:namePart><mods:role><mods:roleTerm type="text">author</mods:roleTerm></mods:role></mods:name><mods:name type="corporate"><mods:namePart>NSF</mods:namePart><mods:role><mods:roleTerm type="text">funder</mods:roleTerm></mods:role></mods:name><mods:name type="corporate"><mods:namePart>CDC</mods:namePart><mods:role><mods:roleTerm type="text">funder</mods:roleTerm></mods:role></mods:name><mods:originInfo><mods:dateIssued encoding="w3cdtf" keyDate="yes">2009</mods:dateIssued></mods:originInfo><mods:genre authority="marcgt">Article</mods:genre><mods:genre authority="local">Preprint: Prior to Peer Review</mods:genre><mods:relatedItem type="otherVersion" displayLabel="Final Published Version"><mods:identifier type="uri" displayLabel="URL">http://www.jstor.org/stable/1852669</mods:identifier><mods:identifier type="doi" displayLabel="DOI">doi:10/1073/pnas/1111088108</mods:identifier></mods:relatedItem><mods:note type="author notes">published under a different name</mods:note><mods:subject authority="keywords"><mods:topic>nature</mods:topic></mods:subject><mods:subject authority="keywords"><mods:topic>biomedical things</mods:topic></mods:subject><mods:subject ID="id0405" authority="proquestresearchfield"><mods:topic>Mathematics</mods:topic></mods:subject><mods:accessCondition type="restrictions on access" displayLabel="RightsNote">The admin note</mods:accessCondition><mods:accessCondition type="restrictions on access" displayLabel="copyrightStatusDeterminationDate">2011-011-11</mods:accessCondition><mods:relatedItem xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://interestingsupportingmaterial.com" type="references" displayLabel="SupplementalMaterial"/></mods:mods>
Do we get here 7???
Do we get here 3???
Do we get here 5???
Do we get here 8???
<oai_dc:dc xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/oai_dc/ http://www.openarchives.org/OAI/2.0/oai_dc.xsd">
  <dc:title>A very scholarly article</dc:title>
  <dc:description>An overly complicated description of a very scholarly article</dc:description>
  <dc:type>text</dc:type>
  <dc:type>Preprint: Prior to Peer Review: article</dc:type>
  <dc:identifier>oe-zo-test:pg17q</dc:identifier>
  <dc:identifier>http://testpid.library.emory.edu/ark:/25593/pg17q</dc:identifier>
  <dc:source>Nature Volume 92 Issue 3 Date 2009 Pages 362-376</dc:source>
<dc:contributor>Minnie Mouse</dc:contributor><dc:contributor>Joe Science</dc:contributor><dc:subject>Mathematics</dc:subject><dc:subject>nature</dc:subject><dc:subject>biomedical things</dc:subject></oai_dc:dc>
Do we get here 7???
Do we get here2???
Do we get here 5???
Do we get here 8???
<mods:mods xmlns:mods="http://www.loc.gov/mods/v3"><mods:titleInfo><mods:title>A very scholarly article</mods:title><mods:partNumber>Part 1</mods:partNumber><mods:partName>Where we are now</mods:partName></mods:titleInfo><mods:abstract>An overly complicated description of a very scholarly article</mods:abstract><mods:relatedItem type="host"><mods:originInfo><mods:publisher>Nature Publishing Group</mods:publisher></mods:originInfo><mods:titleInfo><mods:title>Nature</mods:title></mods:titleInfo><mods:part><mods:detail type="volume"><mods:number>92</mods:number></mods:detail><mods:detail type="number"><mods:number>3</mods:number></mods:detail><mods:extent unit="pages"><mods:start>362</mods:start><mods:end>376</mods:end></mods:extent></mods:part></mods:relatedItem><mods:identifier type="uri">http://testpid.library.emory.edu/ark:/25593/pg17q</mods:identifier><mods:identifier type="ark">ark:/25593/pg17q</mods:identifier><mods:name ID="mmouse" type="personal"><mods:affiliation>Emory University</mods:affiliation><mods:namePart type="family">Mouse</mods:namePart><mods:namePart type="given">Minnie</mods:namePart><mods:role><mods:roleTerm type="text">author</mods:roleTerm></mods:role></mods:name><mods:name type="personal"><mods:affiliation>GA Tech</mods:affiliation><mods:namePart type="given">Joe</mods:namePart><mods:namePart type="family">Science</mods:namePart><mods:role><mods:roleTerm type="text">author</mods:roleTerm></mods:role></mods:name><mods:name type="corporate"><mods:namePart>NSF</mods:namePart><mods:role><mods:roleTerm type="text">funder</mods:roleTerm></mods:role></mods:name><mods:name type="corporate"><mods:namePart>CDC</mods:namePart><mods:role><mods:roleTerm type="text">funder</mods:roleTerm></mods:role></mods:name><mods:originInfo><mods:dateIssued encoding="w3cdtf" keyDate="yes">2009</mods:dateIssued><mods:dateOther type="embargoedUntil" encoding="w3cdtf">2016-08-21</mods:dateOther></mods:originInfo><mods:genre authority="marcgt">Article</mods:genre><mods:genre authority="local">Preprint: Prior to Peer Review</mods:genre><mods:relatedItem type="otherVersion" displayLabel="Final Published Version"><mods:identifier type="uri" displayLabel="URL">http://www.jstor.org/stable/1852669</mods:identifier><mods:identifier type="doi" displayLabel="DOI">doi:10/1073/pnas/1111088108</mods:identifier></mods:relatedItem><mods:note type="author notes">published under a different name</mods:note><mods:subject authority="keywords"><mods:topic>nature</mods:topic></mods:subject><mods:subject authority="keywords"><mods:topic>biomedical things</mods:topic></mods:subject><mods:subject ID="id0405" authority="proquestresearchfield"><mods:topic>Mathematics</mods:topic></mods:subject><mods:accessCondition type="restrictions on access" displayLabel="RightsNote">The admin note</mods:accessCondition><mods:accessCondition type="restrictions on access" displayLabel="copyrightStatusDeterminationDate">2011-011-11</mods:accessCondition><mods:relatedItem xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="http://interestingsupportingmaterial.com" type="references" displayLabel="SupplementalMaterial"/></mods:mods>
Do we get here 7???
]]></system-out></testcase><testcase classname="openemory.publication.tests.PublicationViewsTest" name="test_view_article_biblio" time="2.247"><system-out><![CDATA[Do we get here 3???
Do we get here 5???
Do we get here 8???
<mods:mods xmlns:mods="http://www.loc.gov/mods/v3"><mods:titleInfo><mods:title>A very scholarly article</mods:title><mods:subTitle>A love story</mods:subTitle></mods:titleInfo><mods:abstract>An overly complicated description of a very scholarly article</mods:abstract><mods:relatedItem type="host"><mods:originInfo><mods:publisher>Big Deal Publications</mods:publisher></mods:originInfo><mods:titleInfo><mods:title>Journal of Important Things</mods:title></mods:titleInfo><mods:part><mods:detail type="volume"><mods:number>11</mods:number></mods:detail><mods:detail type="number"><mods:number>5</mods:number></mods:detail><mods:extent unit="pages"><mods:start>1742</mods:start><mods:end>2637</mods:end></mods:extent></mods:part></mods:relatedItem><mods:identifier type="uri">http://testpid.library.emory.edu/ark:/25593/pg18v</mods:identifier><mods:identifier type="ark">ark:/25593/pg18v</mods:identifier><mods:name type="personal"><mods:namePart type="family">Mouse</mods:namePart><mods:namePart type="given">Mickey</mods:namePart><mods:role><mods:roleTerm type="text">author</mods:roleTerm></mods:role></mods:name><mods:name type="personal"><mods:namePart type="family">Mouse</mods:namePart><mods:namePart type="given">Minnie</mods:namePart><mods:role><mods:roleTerm type="text">author</mods:roleTerm></mods:role></mods:name><mods:originInfo><mods:dateIssued encoding="w3cdtf" keyDate="yes">2011-08-24</mods:dateIssued></mods:originInfo><mods:subject authority="keywords"><mods:geographic>Atlanta</mods:geographic></mods:subject><mods:subject authority="keywords"><mods:name><mods:namePart>William</mods:namePart><mods:namePart>Shakespeare</mods:namePart></mods:name></mods:subject><mods:subject authority="keywords"><mods:topic>rumba</mods:topic></mods:subject><mods:subject authority="keywords"><mods:titleInfo><mods:title>Divine Comedy, The</mods:title></mods:titleInfo></mods:subject><mods:relatedItem type="otherVersion" displayLabel="Final Published Version"><mods:identifier type="doi" displayLabel="DOI">doi:42.1234/1-2-3-4</mods:identifier></mods:relatedItem><mods:language><mods:languageTerm type="text">English</mods:languageTerm></mods:language></mods:mods>
Do we get here 7???
Do we get here 3???
Do we get here 5???
Do we get here 8???
<oai_dc:dc xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/oai_dc/ http://www.openarchives.org/OAI/2.0/oai_dc.xsd">
  <dc:title>A very scholarly article: A love story</dc:title>
  <dc:description>An overly complicated description of a very scholarly article</dc:description>
  <dc:type>text</dc:type>
  <dc:type>article</dc:type>
  <dc:identifier>oe-zo-test:pg18v</dc:identifier>
  <dc:identifier>http://testpid.library.emory.edu/ark:/25593/pg18v</dc:identifier>
  <dc:source>Journal of Important Things Volume 11 Issue 5 Date 2011-08-24 Pages 1742-2637</dc:source>
<dc:contributor>Mickey Mouse</dc:contributor><dc:contributor>Minnie Mouse</dc:contributor><dc:language>English</dc:language><dc:subject/><dc:subject/><dc:subject>rumba</dc:subject><dc:subject/></oai_dc:dc>
Do we get here 7???
]]></system-out></testcase><testcase classname="openemory.publication.tests.PublicationViewsTest" name="test_view_article_license" time="2.660"><system-out><![CDATA[Do we get here 3???
Do we get here 5???
Do we get here 8???
<mods:mods xmlns:mods="http://www.loc.gov/mods/v3"><mods:titleInfo><mods:title>A very scholarly article</mods:title></mods:titleInfo><mods:abstract>An overly complicated description of a very scholarly article</mods:abstract><mods:relatedItem type="host"><mods:originInfo><mods:publisher>Big Deal Publications</mods:publisher></mods:originInfo></mods:relatedItem><mods:identifier type="uri">http://testpid.library.emory.edu/ark:/25593/pg190</mods:identifier><mods:identifier type="ark">ark:/25593/pg190</mods:identifier><mods:accessCondition xmlns:ns0="http://www.w3.org/1999/xlink" type="use and reproduction" displayLabel="license" ns0:href="http://creativecommons.org/licenses/by-nc-nd/3.0/">
    Readers may use this
      article as long as the work is properly cited, the use is
      educational and not for profit, and the work is not altered. See
      
     http://creativecommons.org/licenses/by-nc-nd/3.0/
      for details.</mods:accessCondition><mods:accessCondition type="use and reproduction" displayLabel="copyright">(c) 2010 by ADA.</mods:accessCondition></mods:mods>
Do we get here 7???
Do we get here 3???
Do we get here 5???
Do we get here 9???
True
Do we get here 7???
]]></system-out></testcase><testcase classname="openemory.publication.tests.QuarterlyCommandTest" name="test_get_article_data" time="0.007"></testcase><testcase classname="openemory.publication.tests.QuarterlyCommandTest" name="test_send_mail" time="0.010"><system-out><![CDATA[No Mail Sent
Mail Sent
]]></system-out></testcase><testcase classname="openemory.publication.tests.ResearchFieldsTest" name="test_choices" time="0.012"></testcase><testcase classname="openemory.publication.tests.ResearchFieldsTest" name="test_get_choices" time="0.012"></testcase><testcase classname="openemory.publication.tests.ResearchFieldsTest" name="test_init" time="0.000"></testcase><testcase classname="openemory.publication.tests.ResearchFieldsTest" name="test_label" time="0.000"></testcase><testcase classname="openemory.publication.tests.TestExpireEmbargoCommand" name="test_expire_embargo" time="0.000"><skipped type="unittest.case.SkipTest" message="acting differently on local and jenkins"><![CDATA[SkipTest: acting differently on local and jenkins
]]></skipped></testcase><testcase classname="openemory.publication.tests.TestFileTypeValidator" name="test_memory_file" time="0.026"></testcase><testcase classname="openemory.publication.tests.TestFileTypeValidator" name="test_temporary_file" time="0.002"></testcase><testcase classname="openemory.publication.tests.TestSympDS" name="test_basic_fields" time="0.003"></testcase><testcase classname="openemory.publication.tests.TestSympDS" name="test_properties" time="0.383"></testcase><testcase classname="openemory.publication.tests.TestUtil" name="test_percent_match" time="0.001"></testcase><testcase classname="openemory.publication.tests.ValidateNetidTest" name="test_validate_netid" time="0.015"><error type="exceptions.AttributeError" message="assert_called"><![CDATA[Traceback (most recent call last):
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 67, in testPartExecutor
    yield
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/unittest2/case.py", line 625, in run
    testMethod()
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 1305, in patched
    return func(*args, **keywargs)
  File "/Users/azotov/OpenEmory/openemory/publication/tests.py", line 925, in test_validate_netid
    mockldap.return_value.find_user.assert_called
  File "/Users/azotov/VirtualEnv/open-emory/lib/python2.7/site-packages/mock/mock.py", line 721, in __getattr__
    raise AttributeError(name)
AttributeError: assert_called
]]></error></testcase><testcase classname="openemory.harvest.tests.ArticleQuerySetTest" name="test_query_set_requests" time="0.010"></testcase><testcase classname="openemory.harvest.tests.ArticleQuerySetTest" name="test_slicing" time="0.010"></testcase><testcase classname="openemory.harvest.tests.EntrezTest" name="test_get_emory_articles" time="0.015"></testcase><testcase classname="openemory.harvest.tests.FetchPMCMetadataTest" name="test_date_opts" time="0.203"><system-out><![CDATA[Date Range: 2011/09/07 - 2015/08/22
Date Range: 2012/01/01 - 2012/02/02
Date Range: 2011/09/07 - 2015/08/22
]]></system-out></testcase><testcase classname="openemory.harvest.tests.HarvestRecordTest" name="test_as_publication_article" time="0.245"></testcase><testcase classname="openemory.harvest.tests.HarvestRecordTest" name="test_ingestable" time="0.186"></testcase><testcase classname="openemory.harvest.tests.HarvestRecordTest" name="test_init_from_fetched_article" time="0.224"></testcase><testcase classname="openemory.harvest.tests.HarvestRecordTest" name="test_mark_ignored" time="0.194"></testcase><testcase classname="openemory.harvest.tests.HarvestRecordTest" name="test_mark_ingested" time="0.192"></testcase><testcase classname="openemory.harvest.tests.HarvestViewsTest" name="test_queue" time="0.581"></testcase><testcase classname="openemory.harvest.tests.HarvestViewsTest" name="test_queue_ajax" time="0.445"></testcase><testcase classname="openemory.harvest.tests.HarvestViewsTest" name="test_record_delete" time="0.555"></testcase></testsuite>